[
    {
        "error": "'NoneType' object has no attribute 'repository'",
        "issue_number": 16
    },
    {
        "error": "'NoneType' object has no attribute 'repository'",
        "issue_number": 14
    },
    {
        "error": "'NoneType' object has no attribute 'repository'",
        "issue_number": 22
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 17,
        "title": "标注矩形框时报错.",
        "body": "我的环境是win11专业版，python 3.10.10, cpu 版的 paddleocr。运行 paddlelabel 命令如下：\r\n` PPOCRLabel --lang ch --kie True`\r\n \r\n标注矩形框时，报以下错误：\r\nTraceback (most recent call last):\r\n  File \"D:\\myCode\\python\\venv\\lib\\site-packages\\PPOCRLabel\\libs\\canvas.py\", line 596, in paintEvent\r\n    p.drawRect(leftTop.x(), leftTop.y(), rectWidth, rectHeight)\r\nTypeError: arguments did not match any overloaded call:\r\n  drawRect(self, rect: QRectF): argument 1 has unexpected type 'float'\r\n  drawRect(self, x: int, y: int, w: int, h: int): argument 1 has unexpected type 'float'\r\n  drawRect(self, r: QRect): argument 1 has unexpected type 'float'\r\nQBackingStore::endPaint() called with active painter; did you forget to destroy it or call QPainter::end() on it?\r\n\r\n怎么解决呢？",
        "state": "closed",
        "user": "durongzheng",
        "closed_by": "durongzheng",
        "created_at": "2024-06-18T02:26:05+00:00",
        "updated_at": "2024-06-20T14:09:47+00:00",
        "closed_at": "2024-06-18T08:30:14+00:00",
        "comments_count": [
            "GreatV",
            "durongzheng"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 13,
        "title": "PPOCRLabel --lang ch --kie True 下绘制矩形标注框导致程序崩坏",
        "body": "传入坐标信息时，期待传入类型为int，但是传入了float类型。\r\npyANACONDA\\envs\\feijiang_label\\lib\\site-packages\\PPOCRLabel\\PPOCRLabel.py line 1425\r\n原始代码为\r\nbar.setValue(bar.value() + bar.singleStep() * units)\r\n更改后为\r\nbar.setValue(int(bar.value() + bar.singleStep() * units))\r\n\r\n\r\npyANACONDA\\envs\\feijiang_label\\lib\\site-packages\\PPOCRLabel\\libs\\canvas.py line 596\r\n原始代码为\r\np.drawRect(leftTop.x(), leftTop.y(), rectWidth, rectHeight)\r\n更改后为\r\np.drawRect(int(leftTop.x()), int(leftTop.y()), int(rectWidth), int(rectHeight))\r\n\r\n",
        "state": "closed",
        "user": "TTTalen",
        "closed_by": "TTTalen",
        "created_at": "2024-06-05T08:35:55+00:00",
        "updated_at": "2024-06-05T09:00:39+00:00",
        "closed_at": "2024-06-05T08:57:02+00:00",
        "comments_count": [
            "GreatV",
            "TTTalen",
            "GreatV"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 20,
        "title": "PPOCRLabel标注使用中文输入法异常",
        "body": "运行环境 / Runtime Environment\r\npython 3.8\r\npaddlepaddle==2.5.0\r\nPPOCRLabel==2.1.3\r\n\r\n直接从官方文档的环境下载也不行,使用编辑文字秒切英文\r\n\r\nhttps://github.com/PFCCLab/PPOCRLabel/assets/94338396/cbc45bc2-a35f-4a2d-b84d-453f6fe3e71d\r\n\r\n",
        "state": "closed",
        "user": "77bw",
        "closed_by": "GreatV",
        "created_at": "2024-06-24T01:14:33+00:00",
        "updated_at": "2024-06-25T02:51:22+00:00",
        "closed_at": "2024-06-24T12:25:22+00:00",
        "comments_count": [
            "Gmgge",
            "Gmgge",
            "77bw",
            "GreatV",
            "Gmgge"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 21,
        "title": "'utf-8' codec can't decode byte 0xc3 in position 3044: invalid continuation byte",
        "body": "```\r\n PPOCRLabel --lang ch\r\n\r\n[2024/06/28 21:19:02] ppocr WARNING: When args.layout is false, args.ocr is automatically set to false\r\n\r\nTraceback (most recent call last):\r\n  File \"E:\\anaconda3\\envs\\paddlepaddle\\lib\\site-packages\\PPOCRLabel\\PPOCRLabel.py\", line 1737, in openDirDialog\r\n    self.importDirImages(targetDirPath)\r\n  File \"E:\\anaconda3\\envs\\paddlepaddle\\lib\\site-packages\\PPOCRLabel\\PPOCRLabel.py\", line 1794, in importDirImages\r\n    self.loadFilestate(dirpath)\r\n  File \"E:\\anaconda3\\envs\\paddlepaddle\\lib\\site-packages\\PPOCRLabel\\PPOCRLabel.py\", line 2590, in loadFilestate\r\n    states = f.readlines()\r\n  File \"E:\\anaconda3\\envs\\paddlepaddle\\lib\\codecs.py\", line 322, in decode\r\n    (result, consumed) = self._buffer_decode(data, self.errors, final)\r\nUnicodeDecodeError: 'utf-8' codec can't decode byte 0xc3 in position 3044: invalid continuation byte\r\n\r\n```",
        "state": "closed",
        "user": "monkeycc",
        "closed_by": "GreatV",
        "created_at": "2024-06-28T13:22:18+00:00",
        "updated_at": "2024-07-11T14:21:09+00:00",
        "closed_at": "2024-07-11T14:21:09+00:00",
        "comments_count": [
            "GreatV",
            "Unlicensed-driver-ljx",
            "GreatV"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 19,
        "title": "python打开出现问题",
        "body": "[2024/06/20 15:45:19] ppocr WARNING: When args.layout is false, args.ocr is automatically set to false\r\n\r\nValueError: operands could not be broadcast together with shapes (213,488,4) (1,1,3)\r\n\r\n这是咋回事啊",
        "state": "closed",
        "user": "Myles091",
        "closed_by": "GreatV",
        "created_at": "2024-06-20T07:51:19+00:00",
        "updated_at": "2024-07-04T05:27:27+00:00",
        "closed_at": "2024-06-24T12:25:38+00:00",
        "comments_count": [
            "GreatV",
            "Gmgge",
            "Myles091",
            "Myles091",
            "Gmgge",
            "qdhonker"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 15,
        "title": "將PPOCRLabel轉exe後發生ImportError: cannot import name 'DEFAULT_LOCK_COLOR' from 'libs.shape' 。",
        "body": "问题描述 / Problem Description\r\n使用pyinstaller將PPOCRLabel轉exe，生成dist資料夾，內部包含執行檔PPOCRLabel.exe及資料夾_internal。\r\n已參考https://blog.csdn.net/u012254599/article/details/128012719 將所需套件放入後，再將PPOCRLabel-main\\libs放入_internal內。\r\n運行PPOCRLabel.exe發生\r\nImportError: cannot import name 'DEFAULT_LOCK_COLOR' from 'libs.shape' (D:\\OCR\\PaddleOCR env\\PPOCRLabel-main\\dist\\PPOCRLabel\\_internal\\libs\\shape.pyc)。\r\n\r\n运行环境 / Runtime Environment\r\nOS : Win10\r\npaddlepaddle : 2.5.1\r\npaddleocr : 2.6\r\nPaddleOCR : PaddleOCR-release-2.7\r\n\r\n完整报错 / Complete Error Message\r\nTraceback (most recent call last):\r\n  File \"PPOCRLabel-main\\PPOCRLabel.py\", line 77, in <module>\r\nImportError: cannot import name 'DEFAULT_LOCK_COLOR' from 'libs.shape' (D:\\OCR\\PaddleOCR env\\PPOCRLabel-main\\dist\\PPOCRLabel\\_internal\\libs\\shape.pyc)\r\n\r\n求各位大佬幫忙解決!!!",
        "state": "closed",
        "user": "DingHsun",
        "closed_by": "GreatV",
        "created_at": "2024-06-07T06:15:52+00:00",
        "updated_at": "2024-06-18T08:58:50+00:00",
        "closed_at": "2024-06-08T15:49:27+00:00",
        "comments_count": [
            "GreatV",
            "Gmgge"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 12,
        "title": "表格标注 顺序的序号是如何显示的",
        "body": "![image](https://github.com/PFCCLab/PPOCRLabel/assets/94338396/9564ed52-57d2-4bae-8f47-6728a62eb8db)",
        "state": "closed",
        "user": "77bw",
        "closed_by": "GreatV",
        "created_at": "2024-06-03T02:57:14+00:00",
        "updated_at": "2024-06-08T15:49:19+00:00",
        "closed_at": "2024-06-08T15:49:19+00:00",
        "comments_count": [
            "GreatV"
        ],
        "labels": []
    },
    {
        "error": "'NoneType' object has no attribute 'repository'",
        "issue_number": 25
    },
    {
        "error": "'NoneType' object has no attribute 'repository'",
        "issue_number": 30
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 24,
        "title": "The specified module was not found",
        "body": "ImportError: DLL load failed while importing libpaddle: Belirtilen modül bulunamadı.",
        "state": "closed",
        "user": "omeeragtoprak",
        "closed_by": "GreatV",
        "created_at": "2024-07-02T12:10:00+00:00",
        "updated_at": "2024-07-11T14:21:01+00:00",
        "closed_at": "2024-07-11T14:21:01+00:00",
        "comments_count": [
            "Gmgge"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 32,
        "title": "ppocrlabel 一个矩形框标注多行文字，识别效果不理想，怎么解决，只能标注单行文字，然后多行串接在一起么？",
        "body": "我按照github里面的教程，用 ppocrlabel标注身份证，自动标注的时候，文件都是单行标注，比较准确；但是手工把住址，多行标注的时候，重新识别后，大多数情况下，识别的内容都是不准，求解。\r\n我是按照：https://github.com/PFCCLab/PPOCRLabel 这里面的教程安装的，只安装了paddle和 ppocrlabel。\r\n\r\n![image](https://github.com/user-attachments/assets/ada1f727-8b74-4583-bb16-15b189b17343)\r\n\r\n不知道怎么解决，我看到 https://github.com/PaddlePaddle/PaddleOCR/blob/main/applications/%E5%BF%AB%E9%80%9F%E6%9E%84%E5%BB%BA%E5%8D%A1%E8%AF%81%E7%B1%BBOCR.md 这个教程里面，\r\n![image](https://github.com/user-attachments/assets/fef35f27-62c1-40f1-b926-694bfae5fdf4)\r\n\r\n我不知道这是怎么实现的，求老师答疑，谢谢。\r\n我现在困惑的点有两个：一是教程里面，怎么实现的一次标注多行文字；二是，如果只能每次标注单行文字，那后续再把文字串起来，大概怎么个思路串起来啊。请老师开示。",
        "state": "closed",
        "user": "abelezm",
        "closed_by": "abelezm",
        "created_at": "2024-07-17T04:50:39+00:00",
        "updated_at": "2024-07-18T00:16:18+00:00",
        "closed_at": "2024-07-18T00:16:18+00:00",
        "comments_count": [],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 29,
        "title": "标注工具有业界标准吗？如果不统一，大量的劳动力是不是会重复浪费？",
        "body": "标注工具有业界标准吗？如果不统一，大量的劳动力是不是会重复浪费？",
        "state": "closed",
        "user": "350050183",
        "closed_by": null,
        "created_at": "2024-07-13T09:15:44+00:00",
        "updated_at": "2024-07-13T14:46:22+00:00",
        "closed_at": "2024-07-13T14:46:22+00:00",
        "comments_count": [],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 26,
        "title": "qt.qpa.plugin: Could not load the Qt platform plugin \"xcb\"",
        "body": "Can somone please help me understand how to solve this error, seems like some issues with cv2 dependencies, but not sure what exactly is the solution\r\n\r\n**OS:** \r\nubuntu linux\r\n\r\n**modules:**\r\nopencv-contrib-python     4.10.0.84\r\nopencv-python             4.10.0.84\r\nopencv-python-headless    4.10.0.84\r\npaddle2onnx               1.2.4\r\npaddlefsl                 1.1.0\r\npaddlenlp                 2.6.1\r\npaddleocr                 2.8.0\r\npaddlepaddle              2.6.1\r\n\r\n**Function call**\r\n`python PPOCRLabel.py\r\n`\r\n\r\n\r\n\r\n**Error:** \r\nQObject::moveToThread: Current thread (0x55a715570900) is not the object's thread (0x55a718b1d7b0).\r\nCannot move to target thread (0x55a715570900)\r\n\r\nqt.qpa.plugin: Could not load the Qt platform plugin \"xcb\" in \"/home/ubuntu/miniconda3/lib/python3.12/site-packages/cv2/qt/plugins\" even though it was found.\r\nThis application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.\r\n\r\nAvailable platform plugins are: xcb, eglfs, linuxfb, minimal, minimalegl, offscreen, vnc, wayland-egl, wayland, wayland-xcomposite-egl, wayland-xcomposite-glx, webgl.\r\n\r\n\r\n\r\n--------------------------------------\r\nC++ Traceback (most recent call last):\r\n--------------------------------------\r\nNo stack trace in paddle, may be caused by external reasons.\r\n\r\n----------------------\r\nError Message Summary:\r\n----------------------\r\nFatalError: `Process abort signal` is detected by the operating system.\r\n  [TimeInfo: *** Aborted at 1720426730 (unix time) try \"date -d @1720426730\" if you are using GNU date ***]\r\n  [SignalInfo: *** SIGABRT (@0x3e800004ded) received by PID 19949 (TID 0x7f3b4a782740) from PID 19949 ***]\r\n\r\nAborted (core dumped)",
        "state": "closed",
        "user": "EphronM",
        "closed_by": "GreatV",
        "created_at": "2024-07-08T08:24:27+00:00",
        "updated_at": "2024-07-11T14:20:52+00:00",
        "closed_at": "2024-07-11T14:20:52+00:00",
        "comments_count": [
            "Gmgge"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 33,
        "title": "建议提供EXE版本下载",
        "body": "建议提供EXE版本下载\r\n\r\ncpu版\r\ngpu版\r\n\r\n这样就不需要用户去编译了",
        "state": "open",
        "user": "monkeycc",
        "closed_by": null,
        "created_at": "2024-07-17T06:45:03+00:00",
        "updated_at": "2024-12-25T10:21:36+00:00",
        "closed_at": null,
        "comments_count": [
            "GreatV",
            "Gmgge",
            "monkeycc",
            "Gmgge",
            "monkeycc",
            "gewenpulan",
            "LovingThresh",
            "GreatV"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 34,
        "title": "KeyError: 'score'",
        "body": "python 3.10\r\npaddleocr                 2.8.0\r\npaddlepaddle-gpu          2.6.1\r\npefile                    2023.2.7\r\npillow                    10.4.0\r\npip                       24.0\r\nPPOCRLabel                2.1.4\r\n```\r\n\r\npython PPOCRLabel.py --lang ch\r\n\r\n[2024/07/17 15:03:39] ppocr WARNING: When args.layout is false, args.ocr is automatically set to false\r\nTraceback (most recent call last):\r\n  File \"E:\\2023_Code_Paddle\\PPOCRLabel-main\\PPOCRLabel.py\", line 3453, in <module>\r\n    sys.exit(main())\r\n  File \"E:\\2023_Code_Paddle\\PPOCRLabel-main\\PPOCRLabel.py\", line 3448, in main\r\n    app, _win = get_main_app(sys.argv)\r\n  File \"E:\\2023_Code_Paddle\\PPOCRLabel-main\\PPOCRLabel.py\", line 3436, in get_main_app\r\n    win = MainWindow(\r\n  File \"E:\\2023_Code_Paddle\\PPOCRLabel-main\\PPOCRLabel.py\", line 143, in __init__\r\n    result = self.table_ocr(\r\n  File \"D:\\anaconda3\\envs\\ocr\\lib\\site-packages\\paddleocr\\paddleocr.py\", line 855, in __call__\r\n    res, _ = super().__call__(img, return_ocr_result_in_table, img_idx=img_idx)\r\n  File \"D:\\anaconda3\\envs\\ocr\\lib\\site-packages\\paddleocr\\ppstructure\\predict_system.py\", line 172, in __call__\r\n    \"score\": region[\"score\"],\r\nKeyError: 'score'\r\n```",
        "state": "closed",
        "user": "monkeycc",
        "closed_by": "GreatV",
        "created_at": "2024-07-17T07:06:55+00:00",
        "updated_at": "2024-07-18T04:51:02+00:00",
        "closed_at": "2024-07-18T04:51:02+00:00",
        "comments_count": [
            "GreatV",
            "GreatV",
            "monkeycc",
            "GreatV",
            "GreatV",
            "YUECHE77",
            "GreatV",
            "YUECHE77",
            "YUECHE77",
            "GreatV",
            "YUECHE77",
            "YUECHE77",
            "GreatV",
            "GreatV",
            "YUECHE77",
            "GreatV",
            "YUECHE77"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 35,
        "title": "PyInstaller打包报错",
        "body": "```\r\n\r\ncd ./PPOCRLabel\r\n# 安装pyinstaller\r\npip install pyinstaller\r\n\r\n# 重新生成资源\r\npyrcc5 -o libs/resources.py resources.qrc\r\n\r\n# 打包可执行程序\r\npyinstaller -c PPOCRLabel.py --collect-all paddleocr --collect-all pyclipper --collect-all imghdr --collect-all skimage --collect-all imgaug --collect-all scipy.io --collect-all lmdb --collect-all paddle --hidden-import=pyqt5  -p ./libs -p ./ -p ./data -p ./resources -F\r\n\r\n# 运行dist中的可执行程序，以windows为例\r\nPPOCRLabel.exe\r\n```\r\n\r\n```\r\nTraceback (most recent call last):\r\n  File \"PPOCRLabel.py\", line 66, in <module>\r\n  File \"PyInstaller\\loader\\pyimod02_importers.py\", line 419, in exec_module\r\n  File \"paddleocr\\__init__.py\", line 14, in <module>\r\n    from .paddleocr import *\r\n  File \"PyInstaller\\loader\\pyimod02_importers.py\", line 419, in exec_module\r\n  File \"paddleocr\\paddleocr.py\", line 60, in <module>\r\n    from ppocr.utils.network import (\r\n  File \"C:\\Users\\mm\\AppData\\Local\\Temp\\_MEI355082\\paddleocr\\ppocr\\utils\\network.py\", line 18, in <module>\r\n    import requests\r\nModuleNotFoundError: No module named 'requests'\r\n[27212] Failed to execute script 'PPOCRLabel' due to unhandled exception!\r\n```",
        "state": "open",
        "user": "monkeycc",
        "closed_by": null,
        "created_at": "2024-07-17T07:20:11+00:00",
        "updated_at": "2024-12-25T11:38:40+00:00",
        "closed_at": null,
        "comments_count": [
            "GreatV",
            "monkeycc",
            "tlishi"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 36,
        "title": "标注报错,程序退出",
        "body": "PPOCRLabel main 最新版\r\npython 3.10\r\n\r\n多点标注\r\n按Q\r\n\r\n下一张 按的时候\r\n随机触发事件\r\n\r\n```\r\n  File \"D:\\anaconda3\\envs\\ocr\\lib\\site-packages\\PPOCRLabel\\PPOCRLabel.py\", line 1425, in scrollRequest\r\n    bar.setValue(bar.value() + bar.singleStep() * units)\r\nTypeError: setValue(self, a0: int): argument 1 has unexpected type 'float'\r\n```\r\n\r\nctrl + 鼠标中键\r\n```\r\n\r\n  File \"D:\\anaconda3\\envs\\ocr\\lib\\site-packages\\PPOCRLabel\\PPOCRLabel.py\", line 1477, in zoomRequest\r\n    self.addZoom(scale * units)\r\n  File \"D:\\anaconda3\\envs\\ocr\\lib\\site-packages\\PPOCRLabel\\PPOCRLabel.py\", line 1436, in addZoom\r\n    self.setZoom(self.zoomWidget.value() + increment)\r\n  File \"D:\\anaconda3\\envs\\ocr\\lib\\site-packages\\PPOCRLabel\\PPOCRLabel.py\", line 1433, in setZoom\r\n    self.zoomWidget.setValue(value)\r\nTypeError: setValue(self, val: int): argument 1 has unexpected type 'float'\r\n```",
        "state": "closed",
        "user": "monkeycc",
        "closed_by": "GreatV",
        "created_at": "2024-07-17T17:52:15+00:00",
        "updated_at": "2024-07-18T08:23:12+00:00",
        "closed_at": "2024-07-18T04:51:16+00:00",
        "comments_count": [
            "GreatV",
            "monkeycc"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 37,
        "title": "PPOCRLabel 设置了  --kie true，对增值税发票进行要素提取的标注，怎么操作啊，求教程。",
        "body": "PPOCRLabel 有文字OCR的标注教程，也比较好理解。我正在学习 教程里面的增值税发票的教程，不会里面的标注操作，也就是  kie的标注，不知道教程在哪里，我只找到了文字的标注方式的教程，要素提取的木有找到，请问老师，具体的操作步骤在哪里，实现下面这个标注，而且我看有的教程里面，还 有 link 的标注，不知道相关的操作步骤在哪里。\r\n![image](https://github.com/user-attachments/assets/5bd7fb06-baf8-404b-9354-5ecb49451350)\r\n",
        "state": "closed",
        "user": "abelezm",
        "closed_by": "GreatV",
        "created_at": "2024-07-18T00:20:53+00:00",
        "updated_at": "2024-07-23T13:46:33+00:00",
        "closed_at": "2024-07-18T14:10:42+00:00",
        "comments_count": [
            "GreatV",
            "abelezm",
            "abelezm"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 39,
        "title": "导出表格标注无响应，文件夹tableRec_excel_output已创建，但是Excel文件未生成",
        "body": "错误日志\r\nunicodeFilePath is C:\\Users\\admin\\Desktop\\traindata\\701-771\\at06_000702_2.jpg\r\nCan not save excel file, maybe Permission denied (.xlsx is being occupied)\r\nCANNOT OPEN .xlsx. It could be one of the following reasons: Only support Windows | No python win32com\r\nCANNOT OPEN .xlsx. It could be the following reasons: .xlsx is not existed\r\n文件创建成功了，但是Excel无法生成，第一次提示缺少pandas，安装后提示无权限创建\r\n命令框全程使用管理员模式进行操作的\r\n![QQ_1721372738151](https://github.com/user-attachments/assets/8cff653c-d4f7-4164-b95c-850baa509636)\r\n",
        "state": "closed",
        "user": "Duyalun",
        "closed_by": "Duyalun",
        "created_at": "2024-07-19T07:10:22+00:00",
        "updated_at": "2024-07-30T09:14:31+00:00",
        "closed_at": "2024-07-30T09:14:30+00:00",
        "comments_count": [
            "GreatV",
            "Duyalun",
            "Bishow-99",
            "Duyalun"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 40,
        "title": "找了所有的历史issue，也木有啥启发，是手工一行一行的修改吗，或者看那个历史issue，请老师指点。可否给个链接。我发现历史的issue有三个地方， 一是paddocr里面有一部分，二是之前的ppocrlabel有一部分，三是这个板块。三个板块都看了，发现2023年的issue比较多，但是还木有发现可参考的例子。恳盼老师指点。",
        "body": "              找了所有的历史issue，也木有啥启发，是手工一行一行的修改吗，或者看那个历史issue啊，请老师指点。可否给个链接。我发现历史的issue有三个地方， 一是paddocr里面有一部分，二是之前的ppocrlabel有一部分，三是这个板块。三个板块都看了，发现2023您的issue比较多，但是还木有发现可参考的例子。恳盼老师指点。\r\n\r\n_Originally posted by @abelezm in https://github.com/PFCCLab/PPOCRLabel/issues/37#issuecomment-2245292192_\r\n            ",
        "state": "closed",
        "user": "abelezm",
        "closed_by": "abelezm",
        "created_at": "2024-07-23T13:45:45+00:00",
        "updated_at": "2024-07-23T16:59:54+00:00",
        "closed_at": "2024-07-23T16:59:54+00:00",
        "comments_count": [
            "GreatV",
            "abelezm"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 41,
        "title": "打包后无法显示图标",
        "body": "修改libs\\utils.py中的路径\r\ndef newIcon(icon, iconSize=None):\r\n    if iconSize is not None:\r\n        return QIcon(\r\n            # QIcon(__iconpath__ + \"/\" + icon + \".png\").pixmap(iconSize, iconSize)\r\n            QIcon(\":/\" + icon).pixmap(iconSize, iconSize)\r\n        )\r\n    else:\r\n        # return QIcon(__iconpath__ + \"/\" + icon + \".png\")\r\n        return QIcon(\":/\" + icon)",
        "state": "closed",
        "user": "guanfuchen",
        "closed_by": "GreatV",
        "created_at": "2024-07-25T06:50:46+00:00",
        "updated_at": "2024-08-07T00:50:48+00:00",
        "closed_at": "2024-08-07T00:50:48+00:00",
        "comments_count": [
            "GreatV"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 42,
        "title": "对于图片上的文字方向不一致的，打标签时是否要有方向性？",
        "body": "对于图片上的文字方向不一致的，打标签时，选择的方框或4个点，是否要有方向性？",
        "state": "closed",
        "user": "350050183",
        "closed_by": "GreatV",
        "created_at": "2024-07-29T06:24:10+00:00",
        "updated_at": "2024-10-10T02:32:22+00:00",
        "closed_at": "2024-08-07T00:50:40+00:00",
        "comments_count": [
            "GreatV",
            "jg0014"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 46,
        "title": "换行的文字如何识别？手动输入？",
        "body": "\r\n<img width=\"108\" alt=\"屏幕截图 2024-08-05 094606\" src=\"https://github.com/user-attachments/assets/301e0559-b205-4a57-8f7e-130962131750\">\r\n像这种换行的文字，在表格标注时是不是按整个单元格画框，但是这个框里的文字，在重新识别时，换行的只能识别第一行，下面的几行文字就要手动输入么？",
        "state": "closed",
        "user": "PlayerJian",
        "closed_by": "GreatV",
        "created_at": "2024-08-05T01:59:36+00:00",
        "updated_at": "2024-08-07T00:50:28+00:00",
        "closed_at": "2024-08-07T00:50:28+00:00",
        "comments_count": [
            "GreatV",
            "PlayerJian",
            "PlayerJian",
            "GreatV",
            "PlayerJian",
            "PlayerJian",
            "PlayerJian",
            "GreatV",
            "PlayerJian",
            "lwinhong",
            "PlayerJian"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 45,
        "title": "xlrd 升级到2.x 之后，不支持.xlsx",
        "body": "## 问题\r\n  如题所述，升级到2.x之后，表格数据标注无法导出\r\n\r\n  官方有给出说明\r\n   [https://github.com/python-excel/xlrd](url)\r\n   ![image](https://github.com/user-attachments/assets/6e5e9265-1a8a-404d-a4df-667134fe5dda)\r\n\r\n## 解决方案\r\n* 方案1. 降级到1.2.0\r\n* 方案2. 修改源码将生成excel和到处excel的路径后缀，由 ```.xlsx``` 改为 ```.xls```，跑自己改之后的代码\r\n\r\n  -   生成路径代码大致在：\r\n\r\n  [https://github.com/PFCCLab/PPOCRLabel/blob/fba5f8c56dc6a4a3e54645ade08dfe168193982b/PPOCRLabel.py#L2836](url)\r\n    ```python\r\n    excel_path = TableRec_excel_dir + \"{}.xlsx\".format(filename)\r\n   ```\r\n\r\n  -   保存时代码路径大致在:\r\n\r\n     [https://github.com/PFCCLab/PPOCRLabel/blob/fba5f8c56dc6a4a3e54645ade08dfe168193982b/PPOCRLabel.py#L3051](url)\r\n    ```python\r\n      # load csv annotations\r\n      filename, _ = os.path.splitext(os.path.basename(image_path))\r\n      csv_path = os.path.join(TableRec_excel_dir, filename + \".xlsx\")\r\n    ```\r\n  -   如果是之前就已经有```.xlsx``` 格式的excel文件了，就打开直接另存为 ```.xls``` 格式\r\n  \r\n\r\n",
        "state": "closed",
        "user": "lwinhong",
        "closed_by": "lwinhong",
        "created_at": "2024-08-02T09:14:49+00:00",
        "updated_at": "2024-08-06T03:47:08+00:00",
        "closed_at": "2024-08-06T03:37:20+00:00",
        "comments_count": [
            "GreatV",
            "GreatV",
            "lwinhong",
            "lwinhong",
            "GreatV"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 49,
        "title": "多点标注导出不出识别结果",
        "body": "![image](https://github.com/user-attachments/assets/5f8b753b-4520-4597-93b7-6ca800e4b723)\r\n如图，多点标注想获取文本识别结果保存为标注文件，发现只保存了矩形标注的结果，这个怎么保存识别结果呢",
        "state": "open",
        "user": "wzz981",
        "closed_by": null,
        "created_at": "2024-08-09T10:36:31+00:00",
        "updated_at": "2024-09-20T01:55:31+00:00",
        "closed_at": null,
        "comments_count": [
            "PaperTiger-L",
            "wzz981",
            "PaperTiger-L"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 52,
        "title": "导出表格标注无报错，没有弹出excel，gt.txt都是空的，但是显示导出成功",
        "body": "如题",
        "state": "closed",
        "user": "Anthonyfhd",
        "closed_by": "GreatV",
        "created_at": "2024-08-15T02:02:21+00:00",
        "updated_at": "2024-08-21T14:09:01+00:00",
        "closed_at": "2024-08-21T14:09:01+00:00",
        "comments_count": [
            "GreatV",
            "GreatV",
            "Anthonyfhd",
            "Anthonyfhd",
            "Anthonyfhd",
            "Anthonyfhd",
            "Anthonyfhd",
            "Anthonyfhd",
            "GreatV",
            "Anthonyfhd",
            "GreatV",
            "Anthonyfhd",
            "GreatV",
            "Anthonyfhd",
            "GreatV",
            "Anthonyfhd",
            "Anthonyfhd",
            "Anthonyfhd",
            "GreatV",
            "GreatV",
            "Anthonyfhd",
            "Anthonyfhd",
            "joyvw",
            "lihuikenny",
            "GreatV"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 56,
        "title": "ubuntu安装好相关包后，启动失败。ubuntu下有具体python版本和包版本要求文件吗",
        "body": "Got keys from plugin meta data (\"xcb\")\r\nQFactoryLoader::QFactoryLoader() checking directory path \"/home/ubuntu/miniconda3/envs/ppocrlabel/bin/platforms\" ...\r\nloaded library \"/home/ubuntu/miniconda3/envs/ppocrlabel/lib/python3.12/site-packages/cv2/qt/plugins/platforms/libqxcb.so\"\r\nQObject::moveToThread: Current thread (0x1116a50) is not the object's thread (0x63c2d10).\r\nCannot move to target thread (0x1116a50)\r\n\r\nqt.qpa.plugin: Could not load the Qt platform plugin \"xcb\" in \"/home/ubuntu/miniconda3/envs/ppocrlabel/lib/python3.12/site-packages/cv2/qt/plugins\" even though it was found.\r\nThis application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.",
        "state": "closed",
        "user": "kezidanbao",
        "closed_by": "GreatV",
        "created_at": "2024-08-22T09:32:18+00:00",
        "updated_at": "2024-08-23T01:36:02+00:00",
        "closed_at": "2024-08-23T01:36:02+00:00",
        "comments_count": [
            "GreatV",
            "kezidanbao",
            "GreatV",
            "kezidanbao"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 57,
        "title": "表格标注生成的gt.txt文件里面的gt属性值html页面表格样式不匹配的bug",
        "body": "各位老师好，我在标注表格的时候发现一个问题麻烦帮忙解答一下。我标注的是一个简单的表格，在学习完了表格标注培训视频，参考着标注，顺序也调整正确，生成的gt.txt里面的gt的属性值，我复制粘贴到一个txt文本转为html页面查看的时候，发现表格没有对应上样式。我贴一下我上述涉及到的图片和标注文件gt.txt\r\n[gt.txt](https://github.com/user-attachments/files/16755467/gt.txt)\r\n![image](https://github.com/user-attachments/assets/7e30b10f-1736-404e-a6cc-b887072ce188)\r\n![image](https://github.com/user-attachments/assets/20b109a0-4646-4286-ac6c-8c9759c945c3)\r\n我想问一下，这个是不是一个bug？因为gt的属性值对应的是html页面，没有跟表格样式对应上。",
        "state": "closed",
        "user": "freezehe",
        "closed_by": "GreatV",
        "created_at": "2024-08-27T01:56:41+00:00",
        "updated_at": "2024-08-29T03:28:58+00:00",
        "closed_at": "2024-08-29T03:28:58+00:00",
        "comments_count": [
            "GreatV",
            "freezehe",
            "freezehe",
            "freezehe",
            "GreatV",
            "GreatV"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 58,
        "title": "使用whl方式时，报错Qt平台插件无法初始化，请问是需要配置相应的qt环境吗？",
        "body": "pythonw\r\nThis application failed to start because no Qt platform plugin could be\r\ninitialized. Reinstalling the application may fix this problem.\r\n<img width=\"364\" alt=\"ocr_label_bug_qt\" src=\"https://github.com/user-attachments/assets/9e153127-7499-4fb2-b070-ca9c7f0e377c\">\r\n",
        "state": "closed",
        "user": "wangpan0523",
        "closed_by": "GreatV",
        "created_at": "2024-08-27T07:58:37+00:00",
        "updated_at": "2024-08-29T03:29:14+00:00",
        "closed_at": "2024-08-29T03:29:14+00:00",
        "comments_count": [
            "GreatV"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 59,
        "title": "表格识别标准工具安装问题反馈",
        "body": "##  运行硬件信息\r\n- Window10\r\n\r\n##  软件安装\r\n-\t在表格识别ReadMe文档中按照文档说明，安装1.4节数据标注说明中推荐的标注工具[PPOCRLabel]\r\nhttps://github.com/PaddlePaddle/PaddleOCR/blob/main/doc/doc_ch/table_recognition.md（表格识别readme）\r\n\r\n##  问题说明\r\n-\t在Windows环境使用Anaconda按照[PPOCRLabel](https://github.com/PFCCLab/PPOCRLabel/blob/main/README_ch.md)要求安装\r\n安装paddlepaddle==2.6.1\r\n安装pip install PPOCRLabel\r\n查看pip freeze可以看到几个主要安装包如下：\r\nPPOCRLabel==2.1.6\r\npaddleocr==2.8.1\r\npaddlepaddle==2.6.1\r\nopencv-contrib-python==4.10.0.84\r\nopencv-python==4.10.0.84\r\n\r\n然后执行：PPOCRLabel --lang ch  # 启动【普通模式】\r\n报如下错：\r\nOMP: Error #15: Initializing libiomp5md.dll, but found libiomp5md.dll already initialized.\r\nOMP: Hint This means that multiple copies of the OpenMP runtime have been linked into the program. That is dangerous, since it can degrade performance or cause incorrect results. The best thing to do is to ensure that only a single OpenMP runtime is linked into the process, e.g. by avoiding static linking of the OpenMP runtime in any library. As an unsafe, unsupported, undocumented workaround you can set the environment variable KMP_DUPLICATE_LIB_OK=TRUE to allow the program to continue to execute, but that may cause crashes or silently produce incorrect results. For more information, please see http://www.intel.com/software/products/support/.\r\n\r\n## 我觉的是paddleocr==2.8.1、opencv-contrib-python==4.10.0.84、opencv-python==4.10.0.84的版本不匹配的问题，但是在安装ReadMe中并没有写这些包的版本依赖关系。\r\n## 我按照安装步骤安装，安装完PPOCRLabel是2.1.6，paddleocr是2.8.1，而paddleocr-2.8.1依赖的opencv-python是4.10.0.84，而我看论坛里说opencv-python系列包的版本都是4.3以下的，我实在是不能辨别了。\r\n\r\n## 请小伙伴们给一个安装指导，谢谢！\r\n",
        "state": "closed",
        "user": "BotAndyGao",
        "closed_by": "BotAndyGao",
        "created_at": "2024-09-02T10:48:21+00:00",
        "updated_at": "2024-09-03T08:27:05+00:00",
        "closed_at": "2024-09-03T08:27:05+00:00",
        "comments_count": [
            "BotAndyGao"
        ],
        "labels": []
    },
    {
        "error": "'NoneType' object has no attribute 'repository'",
        "issue_number": 75
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 60,
        "title": "按照安装指导安装后执行指定无任何反应，大佬们给指导指导",
        "body": "\r\n## 0. 安装环境\r\n- 硬件Windows 10\r\n- 软件Python 3.10.14\r\n- 运行环境 Anaconda3 \r\n\r\n## 1. 安装与运行\r\n\r\n### 1.1 安装PaddlePaddle\r\n```bash\r\n# 如果您的机器是CPU，请运行以下命令安装\r\npip3 install paddlepaddle==2.6.1 -i https://pypi.tuna.tsinghua.edu.cn/simple\r\n```\r\n\r\n### 1.2 安装与运行PPOCRLabel\r\n\r\nPPOCRLabel可通过whl包与Python脚本两种方式启动，whl包形式启动更加方便，python脚本启动便于二次开发\r\n\r\n#### 1.2.1 通过whl包安装与运行\r\n\r\n##### Windows\r\n\r\n```bash\r\npip3 install PPOCRLabel  # 安装\r\n\r\n# 选择标签模式来启动\r\nPPOCRLabel --lang ch  # 启动【普通模式】，用于打【检测+识别】场景的标签\r\nPPOCRLabel --lang ch --kie True  # 启动 【KIE 模式】，用于打【检测+识别+关键字提取】场景的标签\r\n```\r\n\r\n- 1、执行命令：pip3 install paddlepaddle==2.6.1 -i https://pypi.tuna.tsinghua.edu.cn/simple\r\n```bash\r\n正常安装paddlepaddle==2.6.1\r\n```\r\n- 2、执行命令：pip3 install PPOCRLabel\r\n```shell\r\nInstalling collected packages: pytz, PyQt5-Qt5, pyclipper, lmdb, urllib3, tzdata, termcolor, soupsieve, six, rapidfuzz, pyyaml, PyQt5-sip, pyparsing, packaging, numpy, networkx, more-itertools, lxml, kiwisolver, fonttools, et-xmlfile, cython, cycler, cssselect, colorama, charset-normalizer, cachetools, tqdm, tifffile, shapely, scipy, requests, python-docx, python-dateutil, pyqt5, openpyxl, opencv-python, opencv-contrib-python, lazy-loader, imageio, fire, cssutils, contourpy, beautifulsoup4, scikit-image, premailer, pandas, matplotlib, imgaug, paddleocr, PPOCRLabel\r\n  Attempting uninstall: numpy\r\n    Found existing installation: numpy 2.1.0\r\n    Uninstalling numpy-2.1.0:\r\n      Successfully uninstalled numpy-2.1.0\r\nSuccessfully installed PPOCRLabel-2.1.6 PyQt5-Qt5-5.15.2 PyQt5-sip-12.15.0 beautifulsoup4-4.12.3 cachetools-5.5.0 charset-normalizer-3.3.2 colorama-0.4.6 contourpy-1.3.0 cssselect-1.2.0 cssutils-2.11.1 cycler-0.12.1 cython-3.0.11 et-xmlfile-1.1.0 fire-0.6.0 fonttools-4.53.1 imageio-2.35.1 imgaug-0.4.0 kiwisolver-1.4.5 lazy-loader-0.4 lmdb-1.5.1 lxml-5.3.0 matplotlib-3.9.2 more-itertools-10.4.0 networkx-3.3 numpy-1.26.4 opencv-contrib-python-4.10.0.84 opencv-python-4.10.0.84 openpyxl-3.1.5 packaging-24.1 paddleocr-2.8.1 pandas-2.2.2 premailer-3.10.0 pyclipper-1.3.0.post5 pyparsing-3.1.4 pyqt5-5.15.11 python-dateutil-2.9.0.post0 python-docx-1.1.2 pytz-2024.1 pyyaml-6.0.2 rapidfuzz-3.9.7 requests-2.32.3 scikit-image-0.24.0 scipy-1.14.1 shapely-2.0.6 six-1.16.0 soupsieve-2.6 termcolor-2.4.0 tifffile-2024.8.30 tqdm-4.66.5 tzdata-2024.1 urllib3-2.2.2\r\n```\r\n- 3、执行命令：pip freeze\r\n```shell\r\n(PaddleOCRLabelWhl) E:\\paddle>pip3 freeze\r\nanyio==4.4.0\r\nastor==0.8.1\r\nbeautifulsoup4==4.12.3\r\ncachetools==5.5.0\r\ncertifi==2024.8.30\r\ncharset-normalizer==3.3.2\r\ncolorama==0.4.6\r\ncontourpy==1.3.0\r\ncssselect==1.2.0\r\ncssutils==2.11.1\r\ncycler==0.12.1\r\nCython==3.0.11\r\ndecorator==5.1.1\r\net-xmlfile==1.1.0\r\nexceptiongroup==1.2.2\r\nfire==0.6.0\r\nfonttools==4.53.1\r\nh11==0.14.0\r\nhttpcore==1.0.5\r\nhttpx==0.27.2\r\nidna==3.8\r\nimageio==2.35.1\r\nimgaug==0.4.0\r\nkiwisolver==1.4.5\r\nlazy_loader==0.4\r\nlmdb==1.5.1\r\nlxml==5.3.0\r\nmatplotlib==3.9.2\r\nmore-itertools==10.4.0\r\nnetworkx==3.3\r\nnumpy==1.26.4\r\nopencv-contrib-python==4.10.0.84\r\nopencv-python==4.10.0.84\r\nopenpyxl==3.1.5\r\nopt-einsum==3.3.0\r\npackaging==24.1\r\npaddleocr==2.8.1\r\npaddlepaddle==2.6.1\r\npandas==2.2.2\r\npillow==10.4.0\r\nPPOCRLabel==2.1.6\r\npremailer==3.10.0\r\nprotobuf==3.20.2\r\npyclipper==1.3.0.post5\r\npyparsing==3.1.4\r\nPyQt5==5.15.11\r\nPyQt5-Qt5==5.15.2\r\nPyQt5_sip==12.15.0\r\npython-dateutil==2.9.0.post0\r\npython-docx==1.1.2\r\npytz==2024.1\r\nPyYAML==6.0.2\r\nrapidfuzz==3.9.7\r\nrequests==2.32.3\r\nscikit-image==0.24.0\r\nscipy==1.14.1\r\nshapely==2.0.6\r\nsix==1.16.0\r\nsniffio==1.3.1\r\nsoupsieve==2.6\r\ntermcolor==2.4.0\r\ntifffile==2024.8.30\r\ntqdm==4.66.5\r\ntyping_extensions==4.12.2\r\ntzdata==2024.1\r\nurllib3==2.2.2\r\n```\r\n- 4、执行命令：PPOCRLabel --lang ch\r\n```bash\r\n执行该命令后没有任何反应\r\n```",
        "state": "closed",
        "user": "BotAndyGao",
        "closed_by": "GreatV",
        "created_at": "2024-09-03T03:24:09+00:00",
        "updated_at": "2025-05-06T06:03:15+00:00",
        "closed_at": "2024-09-05T06:28:03+00:00",
        "comments_count": [
            "GreatV",
            "BotAndyGao",
            "BotAndyGao",
            "BotAndyGao",
            "BotAndyGao",
            "GreatV",
            "BotAndyGao",
            "YaoBeiji"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 72,
        "title": "使用PPOCRLabel标注表格后导出的gt.txt文件可以直接用于表格识别训练吗？",
        "body": "## 表格标注后导出的gt.txt文件的数据与用于训练的PubTabNet_2.0.0_train.jsonl 内数据格式不一致？\r\n\r\n差异：\r\n- PubTabNet_2.0.0_train.jsonl数据比导出数据多两个属性：\r\n\"imgid\": 548625\r\n\"split\": \"train\" \r\n- gt.txt文件多一个”gt“属性\r\n\r\n\r\n\r\n### gt.txt数据格式\r\n```shell\r\n{\"filename\": \"6.png\", \"html\": {\"structure\": {\"tokens\": [\"<tbody>\", \"<tr>\", \"<td>\", \"</td>\", \"<td>\", \"</td>\", \"<td>\", \"</td>\", \"<td>\", \"</td>\", \"<td>\", \"</td>\", \"<td>\", \"</td>\", \"<td>\", \"</td>\", \"<td>\", \"</td>\", \"<td>\", \"</td>\", \"<td>\", \"</td>\", \"</tr>\", \"<tr>\", \"<td>\", \"</td>\", \"<td>\", \"</td>\", \"<td>\", \"</td>\", \"<td>\", \"</td>\", \"<td>\", \"</td>\", \"<td>\", \"</td>\", \"<td>\", \"</td>\", \"<td>\", \"</td>\", \"<td>\", \"</td>\", \"<td>\", \"</td>\", \"</tr>\", \"</tbody>\"]}, \"cells\": [{\"tokens\": [\"被\", \"购\", \"买\", \"方\", \"名\", \"称\"], \"bbox\": [[16, 38], [115, 38], [115, 102], [16, 102]]}, {\"tokens\": [\"权\", \"得\", \"点\", \"股\", \"取\", \"时\"], \"bbox\": [[140, 19], [198, 19], [198, 113], [140, 113]]}, {\"tokens\": [\"股\", \"权\", \"取\", \"得\", \"成\", \"本\"], \"bbox\": [[229, 54], [375, 54], [375, 80], [229, 80]]}, {\"tokens\": [\"股\", \"权\", \" \", \"取\", \"得\", \" \", \"比\", \"例\", \" \", \"(\", \"%\", \")\"], \"bbox\": [[406, 6], [464, 6], [464, 125], [406, 125]]}, {\"tokens\": [\"股\", \"权\", \" \", \"取\", \"得\", \" \", \"方\", \"式\"], \"bbox\": [[478, 16], [535, 16], [535, 117], [478, 117]]}, {\"tokens\": [\"购\", \"买\", \"日\"], \"bbox\": [[554, 37], [605, 37], [605, 96], [554, 96]]}, {\"tokens\": [\"购\", \"买\", \"日\", \"的\", \"确\", \"定\", \"依\", \"据\"], \"bbox\": [[625, 17], [704, 17], [704, 119], [625, 119]]}, {\"tokens\": [\"购\", \"买\", \"日\", \"至\", \"期\", \"末\", \"被\", \"购\", \"买\", \"方\", \"的\", \"收\", \"入\"], \"bbox\": [[718, 7], [821, 7], [821, 127], [718, 127]]}, {\"tokens\": [\"购\", \"买\", \"日\", \"至\", \"期\", \"末\", \"被\", \"购\", \"买\", \"方\", \"的\", \"净\", \"利\", \"润\"], \"bbox\": [[831, 10], [931, 10], [931, 127], [831, 127]]}, {\"tokens\": [\"购\", \"买\", \"日\", \"至\", \"期\", \"末\", \"被\", \"购\", \"买\", \"方\", \"的\", \"现\", \"金\", \"流\", \"量\"], \"bbox\": [[945, 7], [1046, 7], [1046, 126], [945, 126]]}, {\"tokens\": [\"上\", \"海\", \"凤\", \"凰\", \"驿\", \"站\", \"体\", \"育\", \"文\", \"化\", \"发\", \"展\", \"有\", \"限\", \"公\", \"司\"], \"bbox\": [[13, 133], [120, 133], [120, 254], [13, 254]]}, {\"tokens\": [\"2\", \"0\", \"2\", \"3\", \" \", \"年\", \"1\", \"2\", \" \", \"月\", \"2\", \"7\", \" \", \"日\"], \"bbox\": [[126, 131], [205, 131], [205, 254], [126, 254]]}, {\"tokens\": [\"1\", \"9\", \"5\", \",\", \"5\", \"3\", \"0\", \".\", \"5\", \"4\"], \"bbox\": [[269, 136], [389, 136], [389, 158], [269, 158]]}, {\"tokens\": [\"5\", \"0\", \".\", \"0\", \"0\"], \"bbox\": [[403, 136], [468, 136], [468, 160], [403, 160]]}, {\"tokens\": [\"现\", \"金\"], \"bbox\": [[476, 133], [529, 133], [529, 161], [476, 161]]}, {\"tokens\": [\"2\", \"0\", \"2\", \"3\", \" \", \"年\", \" \", \"1\", \"2\", \" \", \"月\", \" \", \"2\", \"7\", \" \", \"日\"], \"bbox\": [[543, 133], [616, 133], [616, 250], [543, 250]]}, {\"tokens\": [\"对\", \"被\", \"合\", \"并\", \"方\", \"拥\", \"有\", \"实\", \"质\", \"性\", \"权\", \"力\"], \"bbox\": [[627, 133], [707, 133], [707, 252], [627, 252]]}, {\"tokens\": [], \"bbox\": [[713, 135], [822, 135], [822, 253], [713, 253]]}, {\"tokens\": [], \"bbox\": [[831, 136], [940, 136], [940, 254], [831, 254]]}, {\"tokens\": [], \"bbox\": [[946, 137], [1041, 137], [1041, 254], [946, 254]]}]}, \"gt\": \"<html><body><table><tbody><tr><td>被购买方名称</td><td>权得点股取时</td><td>股权取得成本</td><td>股权 取得 比例 (%)</td><td>股权 取得 方式</td><td>购买日</td><td>购买日的确定依据</td><td>购买日至期末被购买方的收入</td><td>购买日至期末被购买方的净利润</td><td>购买日至期末被购买方的现金流量</td></tr><tr><td>上海凤凰驿站体育文化发展有限公司</td><td>2023 年12 月27 日</td><td>195,530.54</td><td>50.00</td><td>现金</td><td>2023 年 12 月 27 日</td><td>对被合并方拥有实质性权力</td><td></td><td></td><td></td></tr></tbody></table></body></html>\"}\r\n\r\n```\r\n###  PubTabNet_2.0.0_train.jsonl 格式数据\r\n```shell\r\n{\"imgid\": 548625, \"html\": {\"cells\": [{\"tokens\": []}, {\"tokens\": [\"<b>\", \"W\", \"e\", \"a\", \"n\", \"i\", \"n\", \"g\", \"</b>\"], \"bbox\": [66, 4, 96, 13]}, {\"tokens\": [\"<b>\", \"W\", \"e\", \"e\", \"k\", \" \", \"1\", \"5\", \"</b>\"], \"bbox\": [131, 4, 160, 13]}, {\"tokens\": [\"<b>\", \"O\", \"f\", \"f\", \"-\", \"t\", \"e\", \"s\", \"t\", \"</b>\"], \"bbox\": [201, 4, 226, 13]}, {\"tokens\": [\"W\", \"e\", \"a\", \"n\", \"i\", \"n\", \"g\"], \"bbox\": [1, 17, 31, 26]}, {\"tokens\": [\"–\"], \"bbox\": [66, 21, 72, 25]}, {\"tokens\": [\"–\"], \"bbox\": [131, 21, 137, 25]}, {\"tokens\": [\"–\"], \"bbox\": [201, 21, 207, 25]}, {\"tokens\": [\"W\", \"e\", \"e\", \"k\", \" \", \"1\", \"5\"], \"bbox\": [1, 31, 30, 40]}, {\"tokens\": [\"–\"], \"bbox\": [66, 35, 72, 39]}, {\"tokens\": [\"0\", \".\", \"1\", \"7\", \" \", \"±\", \" \", \"0\", \".\", \"0\", \"8\"], \"bbox\": [131, 31, 166, 40]}, {\"tokens\": [\"0\", \".\", \"1\", \"6\", \" \", \"±\", \" \", \"0\", \".\", \"0\", \"3\"], \"bbox\": [201, 31, 236, 40]}, {\"tokens\": [\"O\", \"f\", \"f\", \"-\", \"t\", \"e\", \"s\", \"t\"], \"bbox\": [1, 45, 26, 54]}, {\"tokens\": [\"–\"], \"bbox\": [66, 49, 72, 53]}, {\"tokens\": [\"0\", \".\", \"8\", \"0\", \" \", \"±\", \" \", \"0\", \".\", \"2\", \"4\"], \"bbox\": [131, 45, 166, 54]}, {\"tokens\": [\"0\", \".\", \"1\", \"9\", \" \", \"±\", \" \", \"0\", \".\", \"0\", \"9\"], \"bbox\": [201, 45, 236, 54]}], \"structure\": {\"tokens\": [\"<thead>\", \"<tr>\", \"<td>\", \"</td>\", \"<td>\", \"</td>\", \"<td>\", \"</td>\", \"<td>\", \"</td>\", \"</tr>\", \"</thead>\", \"<tbody>\", \"<tr>\", \"<td>\", \"</td>\", \"<td>\", \"</td>\", \"<td>\", \"</td>\", \"<td>\", \"</td>\", \"</tr>\", \"<tr>\", \"<td>\", \"</td>\", \"<td>\", \"</td>\", \"<td>\", \"</td>\", \"<td>\", \"</td>\", \"</tr>\", \"<tr>\", \"<td>\", \"</td>\", \"<td>\", \"</td>\", \"<td>\", \"</td>\", \"<td>\", \"</td>\", \"</tr>\", \"</tbody>\"]}}, \"split\": \"val\", \"filename\": \"PMC5755158_010_01.png\"}\r\n```\r\n\r\n\r\n##  我们使用什么方法，进行表格识别数据的数据切分？\r\n",
        "state": "closed",
        "user": "BotAndyGao",
        "closed_by": "BotAndyGao",
        "created_at": "2024-09-10T08:14:01+00:00",
        "updated_at": "2024-09-11T13:28:34+00:00",
        "closed_at": "2024-09-11T13:28:34+00:00",
        "comments_count": [],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 61,
        "title": "libs.resources 貌似没提交 报错newIcon",
        "body": "如题：\r\n![image](https://github.com/user-attachments/assets/435bfcdb-a2f5-460a-a4df-a3ddc22e9d84)\r\n",
        "state": "closed",
        "user": "lwinhong",
        "closed_by": "GreatV",
        "created_at": "2024-09-04T07:22:32+00:00",
        "updated_at": "2024-09-05T06:28:04+00:00",
        "closed_at": "2024-09-05T06:28:03+00:00",
        "comments_count": [
            "GreatV"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 62,
        "title": "编译完无法启动",
        "body": "编译完成后\r\n![image](https://github.com/user-attachments/assets/aafcaaa1-e2d5-426f-991b-368d05fdd5a6)\r\n\r\n![image](https://github.com/user-attachments/assets/36332fee-a5d3-4d0b-812d-30731f9c15c6)\r\n\r\n运行 PPOCRLabel.exe --lang ch，鼠标转了会圈就没有任何反应了\r\n",
        "state": "closed",
        "user": "Andyszl",
        "closed_by": "GreatV",
        "created_at": "2024-09-05T01:44:56+00:00",
        "updated_at": "2024-09-05T06:28:04+00:00",
        "closed_at": "2024-09-05T06:28:04+00:00",
        "comments_count": [
            "GreatV",
            "Andyszl",
            "GreatV"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 84,
        "title": "MacBook Pro M2 macOS15使用自动标注会卡死",
        "body": "硬件：MacBook Pro M2\r\n系统：15.1 Beta版(24B5046f)\r\n\r\nPPOCRLabel --lang ch                                                                                                                                                                                                                                       \r\n[2024/09/18 22:39:31] ppocr WARNING: When args.layout is false, args.ocr is automatically set to false\r\n2024-09-18 22:39:33.539 python3[97445:1704000] +[IMKClient subclass]: chose IMKClient_Modern\r\n2024-09-18 22:39:33.539 python3[97445:1704000] +[IMKInputSession subclass]: chose IMKInputSession_Modern\r\nfile name in openNext is  /Users/olivia/MyObjects/2024Projects/temps/WechatIMG170.jpg\r\nDirPath in importDirImages is /Users/olivia/MyObjects/2024Projects/temps\r\nloadShapes\r\nloadShapes\r\nUsing model from  paddle",
        "state": "open",
        "user": "Realybig",
        "closed_by": null,
        "created_at": "2024-09-18T14:56:34+00:00",
        "updated_at": "2024-09-23T11:12:55+00:00",
        "closed_at": null,
        "comments_count": [
            "Realybig",
            "GreatV",
            "Realybig",
            "GreatV"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 88,
        "title": "Windows自动保存未提交变更卡死，其他平台应该也有这个问题",
        "body": "卡死情况描述：\r\n1、要启用自动保存未提交变更\r\n2、如果过程中使用删除图像功能\r\n3、会导致卡死\r\n初步怀疑是删除图像后未更新自动保存列表导致索引不存在导致的bug",
        "state": "closed",
        "user": "houjiaxxin",
        "closed_by": "yes-github",
        "created_at": "2024-09-22T09:08:53+00:00",
        "updated_at": "2024-10-31T04:41:47+00:00",
        "closed_at": "2024-10-31T04:41:46+00:00",
        "comments_count": [
            "GreatV",
            "yes-github"
        ],
        "labels": [
            "bug"
        ]
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 89,
        "title": "建议恢复之前对于已经提交保存的标注列表左侧的对号提示",
        "body": "建议恢复之前对于已经提交保存的标注列表左侧的对号提示，这样使用起来会很方便。",
        "state": "closed",
        "user": "houjiaxxin",
        "closed_by": "yes-github",
        "created_at": "2024-09-22T09:21:42+00:00",
        "updated_at": "2024-10-31T04:40:49+00:00",
        "closed_at": "2024-10-31T04:40:49+00:00",
        "comments_count": [
            "yes-github"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 90,
        "title": "kie中的训练re功能如何标注，ppocrlabel中标注的没有re功能需要的id和link字段，并且label字段导出来是key_cls，规则上应该是label",
        "body": "kie中的训练re功能如何标注，ppocrlabel中标注的没有re功能需要的id和link字段，并且label字段导出来是key_cls，规则上应该是label",
        "state": "open",
        "user": "PlayerJian",
        "closed_by": null,
        "created_at": "2024-09-24T01:08:21+00:00",
        "updated_at": "2024-11-02T14:28:02+00:00",
        "closed_at": null,
        "comments_count": [
            "Mhw-git",
            "GreatV",
            "metoogo",
            "cwmore"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 92,
        "title": "不规则合并该如何标注？",
        "body": "<img width=\"396\" alt=\"image\" src=\"https://github.com/user-attachments/assets/33c2d779-56d6-4a4a-b036-f31ed5226377\">\r\n\r\n规则的要补全和顺序排列我明白，表格宽度不规则，不知道如何标注。",
        "state": "open",
        "user": "zhugexinxin",
        "closed_by": null,
        "created_at": "2024-10-01T14:48:47+00:00",
        "updated_at": "2024-10-31T04:43:10+00:00",
        "closed_at": null,
        "comments_count": [
            "yes-github"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 91,
        "title": "Linux issued when run PPOCRLabel",
        "body": "If you guy have install Paddle and PPOCRLabel on linux and have this issued.\r\n![2024-09-29_13-25](https://github.com/user-attachments/assets/29919efd-45de-4af7-98c2-f77a14a1b9b8)\r\nyou should try unistall opencv-python and install opencv-python-headless\r\n\r\nlike this \r\n$ pip uninstalll opencv-python\r\n$ pip install opencv-python-headless\r\n\r\nThat work well for me, thanks\r\n",
        "state": "closed",
        "user": "audihop354",
        "closed_by": "GreatV",
        "created_at": "2024-09-29T06:30:45+00:00",
        "updated_at": "2024-10-27T03:49:36+00:00",
        "closed_at": "2024-10-27T03:49:36+00:00",
        "comments_count": [
            "audihop354",
            "blizaga"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 93,
        "title": "qt.qpa.plugin: Could not load the Qt platform plugin \"xcb\" in \"\" even though it was found.",
        "body": "If you get a warnings like that when trying to run on ubuntu:\n![Image](https://github.com/user-attachments/assets/d97818f0-90b5-41ba-ae8b-7eddc9518626)\nDefinitely you miss some lib in your OS. Try running the following command:\n``` sudo apt install libxcb-xinerama0 ```",
        "state": "closed",
        "user": "phnguynmkhoi",
        "closed_by": "GreatV",
        "created_at": "2024-10-04T03:33:44+00:00",
        "updated_at": "2024-10-29T03:03:27+00:00",
        "closed_at": "2024-10-27T03:49:20+00:00",
        "comments_count": [
            "audihop354",
            "phnguynmkhoi",
            "Jiang7Ting",
            "GreatV",
            "Jiang7Ting"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 94,
        "title": "能否提供re模型的标注方式",
        "body": "ppocrLabel无法完成re模型的标注，缺少id和link字段",
        "state": "open",
        "user": "ybyghuber",
        "closed_by": null,
        "created_at": "2024-10-21T05:30:53+00:00",
        "updated_at": "2025-06-19T22:59:58+00:00",
        "closed_at": null,
        "comments_count": [
            "GreatV",
            "361292623",
            "GreatV",
            "361292623",
            "aymennasri"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 95,
        "title": "Apple m2 pro运行后无法打开窗口，程序没有响应",
        "body": "版本信息\nmacos：15.0.1 (24A348)\npython：3.9.6\npaddleocr：2.9.0\npaddlepaddle：2.6.1\nPyQt5：5.15.11\n\n问题截图\n![Image](https://github.com/user-attachments/assets/68ec845c-2d60-4f24-aa6f-f7fe4b81bb11)\n",
        "state": "closed",
        "user": "idonthavename",
        "closed_by": "GreatV",
        "created_at": "2024-10-22T01:55:43+00:00",
        "updated_at": "2024-10-27T03:43:20+00:00",
        "closed_at": "2024-10-27T03:43:19+00:00",
        "comments_count": [
            "GreatV",
            "idonthavename",
            "GreatV",
            "idonthavename",
            "GreatV"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 96,
        "title": "建议给出PPOCRLabel安装说明",
        "body": "目前因用 python 3.9.0   python 3.10 ，自动安装PPOCRLabel后使用时出现异常：\n\npython 3.9.0 ：\n      initPos PyQt5.QtCore.QPointF(1203.0, 279.8076923076923)   Can not recognise the box\n\n\npython 3.10 \n\n \"c:\\ProgramData\\miniconda3\\envs\\paddlex\\Lib\\site-packages\\PPOCRLabel\\libs\\canvas.py\", line 587, in paintEvent\n    shape.paint(p)\n  File \"c:\\ProgramData\\miniconda3\\envs\\paddlex\\Lib\\site-packages\\PPOCRLabel\\libs\\shape.py\", line 183, in paint\n    painter.drawText(min_x, min_y, self.label)\nTypeError: arguments did not match any overloaded call:\n  drawText(self, p: Union[QPointF, QPoint], s: Optional[str]): argument 1 has unexpected type 'float'\n  drawText(self, rectangle: QRectF, flags: int, text: Optional[str]): argument 1 has unexpected type 'float'\n  drawText(self, rectangle: QRect, flags: int, text: Optional[str]): argument 1 has unexpected type 'float'\n  drawText(self, rectangle: QRectF, text: Optional[str], option: QTextOption = QTextOption()): argument 1 has unexpected type 'float'\n  drawText(self, p: QPoint, s: Optional[str]): argument 1 has unexpected type 'float'\n  drawText(self, x: int, y: int, width: int, height: int, flags: int, text: Optional[str]): argument 1 has unexpected type 'float'\n  drawText(self, x: int, y: int, s: Optional[str]): argument 1 has unexpected type 'float'\nQBackingStore::endPaint() called with active painter; did you forget to destroy it or call QPainter::end() on it?\nQPainter::begin: A paint device can only be painted by one painter at a time.\nQPainter::translate: Painter not active\nQPainter::worldTransform: Painter not active\nQWidgetEffectSourcePrivate::pixmap: Painter not active\nQPainter::save: Painter not active\nQPainter::setOpacity: Painter not active\nQPainter::setWorldTransform: Painter not active\nQPainter::restore: Unbalanced save/restore\nQPainter::begin: Painter already active\n\n\n建议给出安装说明，谢谢\n\n      ",
        "state": "closed",
        "user": "UWPStone",
        "closed_by": "GreatV",
        "created_at": "2024-10-23T15:09:54+00:00",
        "updated_at": "2024-10-27T05:19:47+00:00",
        "closed_at": "2024-10-27T05:19:47+00:00",
        "comments_count": [
            "GreatV",
            "GreatV",
            "UWPStone",
            "GreatV",
            "UWPStone",
            "GreatV",
            "UWPStone",
            "GreatV",
            "UWPStone",
            "GreatV"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 97,
        "title": "python version for PPOCRLabel",
        "body": "Please provide instructions for installing the correct python version in the README section, the supported version for PPOCRLabel is currently only for python version 3.8.",
        "state": "closed",
        "user": "blizaga",
        "closed_by": "blizaga",
        "created_at": "2024-10-24T00:08:51+00:00",
        "updated_at": "2024-10-24T22:32:27+00:00",
        "closed_at": "2024-10-24T22:32:25+00:00",
        "comments_count": [
            "GreatV",
            "blizaga"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 103,
        "title": "关于表格标识时顺序的调整问题【有点让人抓狂的感觉】",
        "body": "关于顺序的调整问题，如果有手动矩形标识的的数量有点多，调整顺序非常的麻烦哦，这个手动调整的工作量很大呢，让人抓狂。\n我都矩形标识出来了，请问程序可以自动按照从上到下，从左到右地调整 顺序吗？可以考虑下。不然，真的没有办法使用这个标注工具了。太痛苦了。\n\n",
        "state": "closed",
        "user": "51MarkDenker65080174",
        "closed_by": "GreatV",
        "created_at": "2024-11-03T16:21:40+00:00",
        "updated_at": "2025-06-14T10:37:42+00:00",
        "closed_at": "2025-06-14T10:37:42+00:00",
        "comments_count": [
            "BotAndyGao"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 102,
        "title": "bug以及优化建议反馈",
        "body": "1、最新版本的Re-recognition功能按钮没有了\n2、删除图片后会重置到第一个图片，这个是有问题的，比如我现在正在操作1500张图片，删除图片后，跳转到第一张，又得挨个找原来的进度，很麻烦\n3、识别自动保存的图片在左侧列表中不会变为绿色对号图标，这个希望能优化为跟手动点确认一样对号提示。",
        "state": "open",
        "user": "houjiaxxin",
        "closed_by": null,
        "created_at": "2024-11-03T01:22:47+00:00",
        "updated_at": "2024-11-21T03:14:01+00:00",
        "closed_at": null,
        "comments_count": [
            "51MarkDenker65080174",
            "yes-github"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 104,
        "title": "Bugs及优化建议收集（Bugs and Improvement Suggestions Collection）",
        "body": "欢迎大家在此issue下提交bug和优化建议（Feel free to submit bugs and suggestions for improvements under this issue）\n\n1. 最新版本的Re-recognition功能按钮没有了\n2. 删除图片后会重置到第一个图片，这个是有问题的，比如我现在正在操作1500张图片，删除图片后，跳转到第一张，又得挨个找原来的进度，很麻烦\n3. 识别自动保存的图片在左侧列表中不会变为绿色对号图标，这个希望能优化为跟手动点确认一样对号提示。\n4. 表格标注用户体验增强，关于顺序的调整问题，如果有手动矩形标识的的数量有点多，调整顺序非常的麻烦哦，这个手动调整的工作量很大呢，让人抓狂。\n我都矩形标识出来了，请问程序可以自动按照从上到下，从左到右地调整 顺序吗？可以考虑下。不然，真的没有办法使用这个标注工具了。太痛苦了。\n5. KIE模式用户体验优化，ppocrLabel无法完成re模型的标注，缺少id和link字段。\n\n\n- #102 \n- #103 \n- #94",
        "state": "open",
        "user": "GreatV",
        "closed_by": null,
        "created_at": "2024-11-04T01:08:41+00:00",
        "updated_at": "2025-06-19T12:50:28+00:00",
        "closed_at": null,
        "comments_count": [
            "361292623",
            "BUJIQI",
            "muyanchen-zen",
            "hotpot-killer",
            "Sam-gsj",
            "aymennasri"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 105,
        "title": "Cam't run PPOCRLabel - illegal instruction",
        "body": "After installing `PPOCRLabel` via `pip` I get the following error when trying to launch the application:\n\n```\n--------------------------------------\nC++ Traceback (most recent call last):\n--------------------------------------\n0   paddle_infer::Predictor::Predictor(paddle::AnalysisConfig const&)\n1   std::unique_ptr<paddle::PaddlePredictor, std::default_delete<paddle::PaddlePredictor> > paddle::CreatePaddlePredictor<paddle::AnalysisConfig, (paddle::PaddleEngineKind)2>(paddle::AnalysisConfig const&)\n2   paddle::AnalysisPredictor::Init(std::shared_ptr<paddle::framework::Scope> const&, std::shared_ptr<paddle::framework::ProgramDesc> const&)\n3   paddle::AnalysisPredictor::PrepareProgram(std::shared_ptr<paddle::framework::ProgramDesc> const&)\n4   paddle::AnalysisPredictor::OptimizeInferenceProgram()\n5   paddle::inference::analysis::Analyzer::RunAnalysis(paddle::inference::analysis::Argument*)\n6   paddle::inference::analysis::IrAnalysisPass::RunImpl(paddle::inference::analysis::Argument*)\n7   paddle::inference::analysis::IRPassManager::Apply(std::unique_ptr<paddle::framework::ir::Graph, std::default_delete<paddle::framework::ir::Graph> >)\n8   paddle::framework::ir::Pass::Apply(paddle::framework::ir::Graph*) const\n9   paddle::framework::ir::SelfAttentionFusePass::ApplyImpl(paddle::framework::ir::Graph*) const\n10  paddle::framework::ir::GraphPatternDetector::operator()(paddle::framework::ir::Graph*, std::function<void (std::map<paddle::framework::ir::PDNode*, paddle::framework::ir::Node*, paddle::framework::ir::GraphPatternDetector::PDNodeCompare, std::allocator<std::pair<paddle::framework::ir::PDNode* const, paddle::framework::ir::Node*> > > const&, paddle::framework::ir::Graph*)>)\n\n----------------------\nError Message Summary:\n----------------------\nFatalError: `Illegal instruction` is detected by the operating system.\n  [TimeInfo: *** Aborted at 1730751475 (unix time) try \"date -d @1730751475\" if you are using GNU date ***]\n  [SignalInfo: *** SIGILL (@0x79c668e5a13a) received by PID 86765 (TID 0x79c6a5bd6000) from PID 1759879482 ***]\n\nIllegal instruction (core dumped)\n```\n\nAny ideas?",
        "state": "closed",
        "user": "haimat",
        "closed_by": "haimat",
        "created_at": "2024-11-04T20:20:14+00:00",
        "updated_at": "2024-11-12T14:49:03+00:00",
        "closed_at": "2024-11-12T14:49:03+00:00",
        "comments_count": [
            "GreatV",
            "haimat"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 106,
        "title": "关于自定义图像的表格识别数据集标注方式",
        "body": "![Image](https://github.com/user-attachments/assets/e8291c25-00e4-4a77-ba21-2397ba0b72a3)\n\n老师您好，我使用PPOCRLabel标注自己的表格识别数据集时，生成的gt.txt文件中的html一直不正确，想向您请教下我这类图片的标准顺序，期待您的回复\n",
        "state": "open",
        "user": "GuoQingru0911",
        "closed_by": null,
        "created_at": "2024-11-07T08:19:19+00:00",
        "updated_at": "2024-11-26T09:24:02+00:00",
        "closed_at": null,
        "comments_count": [
            "BotAndyGao"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 107,
        "title": "PPocrlabel windows 批量自动标注图片时报错",
        "body": "Traceback (most recent call last):\n  File \"d:\\dev_soft\\Anaconda3\\envs\\ppocr\\Lib\\site-packages\\PPOCRLabel\\libs\\autoDialog.py\", line 43, in run\n    self.result_dic = self.ocr.ocr(Imgpath, cls=True, det=True)[0]\n  File \"D:\\dev_soft\\Anaconda3\\envs\\ppocr\\lib\\site-packages\\paddleocr\\paddleocr.py\", line 760, in ocr\n    dt_boxes, rec_res, _ = self.__call__(img, cls, slice)\n  File \"D:\\dev_soft\\Anaconda3\\envs\\ppocr\\lib\\site-packages\\paddleocr\\tools\\infer\\predict_system.py\", line 109, in __call__\n    dt_boxes, elapse = self.text_detector(img)\n  File \"D:\\dev_soft\\Anaconda3\\envs\\ppocr\\lib\\site-packages\\paddleocr\\tools\\infer\\predict_det.py\", line 396, in __call__\n    dt_boxes, elapse = self.predict(img)\n  File \"D:\\dev_soft\\Anaconda3\\envs\\ppocr\\lib\\site-packages\\paddleocr\\tools\\infer\\predict_det.py\", line 255, in predict\n    self.predictor.run()\nRuntimeError: could not create a primitive\n总是跑一半就这样子，windowscpu",
        "state": "open",
        "user": "Keven-Don",
        "closed_by": null,
        "created_at": "2024-11-11T15:45:08+00:00",
        "updated_at": "2024-11-26T09:21:11+00:00",
        "closed_at": null,
        "comments_count": [
            "BotAndyGao"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 108,
        "title": "关于系统提示FatalError: `Process abort signal` is detected by the operating system.",
        "body": "我发现我在启动PPOCRLabel时，出现了如下情况\n```\nGot keys from plugin meta data (\"xcb\")\nQFactoryLoader::QFactoryLoader() checking directory path \"/home/Mashiro/.pyenv/versions/3.10.10/bin/platforms\" ...\nqt.qpa.plugin: Could not find the Qt platform plugin \"dxcb\" in \"/media/Mashiro/inst_47/Project/python/PPOCRLabel/.venv/lib/python3.10/site-packages/cv2/qt/plugins\"\nloaded library \"/media/Mashiro/inst_47/Project/python/PPOCRLabel/.venv/lib/python3.10/site-packages/cv2/qt/plugins/platforms/libqxcb.so\"\nQObject::moveToThread: Current thread (0x55a96abfda50) is not the object's thread (0x55a9704e3da0).\nCannot move to target thread (0x55a96abfda50)\n\nqt.qpa.plugin: Could not load the Qt platform plugin \"xcb\" in \"/media/Mashiro/inst_47/Project/python/PPOCRLabel/.venv/lib/python3.10/site-packages/cv2/qt/plugins\" even though it was found.\nThis application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.\n\nAvailable platform plugins are: xcb, linuxfb, minimal, offscreen, vnc, webgl.\n\n\n\n--------------------------------------\nC++ Traceback (most recent call last):\n--------------------------------------\nNo stack trace in paddle, may be caused by external reasons.\n\n----------------------\nError Message Summary:\n----------------------\nFatalError: `Process abort signal` is detected by the operating system.\n  [TimeInfo: *** Aborted at 1731403692 (unix time) try \"date -d @1731403692\" if you are using GNU date ***]\n  [SignalInfo: *** SIGABRT (@0x3e80001ebcb) received by PID 125899 (TID 0x7f1700863b80) from PID 125899 ***]\n\n\n```\n然后在pip中的依赖如下列所示\n```\nalbucore==0.0.13\nalbumentations==1.4.10\nannotated-types==0.7.0\nanyio==4.6.2.post1\nastor==0.8.1\nbeautifulsoup4==4.12.3\ncachetools==5.5.0\ncertifi==2024.8.30\ncharset-normalizer==3.4.0\ncontourpy==1.3.0\ncssselect==1.2.0\ncssutils==2.11.1\ncycler==0.12.1\nCython==3.0.11\ndecorator==5.1.1\net_xmlfile==2.0.0\nexceptiongroup==1.2.2\nfire==0.7.0\nfonttools==4.54.1\nh11==0.14.0\nhttpcore==1.0.6\nhttpx==0.27.2\nidna==3.10\nimageio==2.36.0\nimgaug==0.4.0\njoblib==1.4.2\nkiwisolver==1.4.7\nlazy_loader==0.4\nlmdb==1.5.1\nlxml==5.3.0\nmatplotlib==3.9.2\nmore-itertools==10.5.0\nnetworkx==3.4.2\nnumpy==1.26.4\nopencv-contrib-python==4.10.0.84\nopencv-python==4.10.0.84\nopencv-python-headless==4.10.0.84\nopenpyxl==3.1.5\nopt-einsum==3.3.0\npackaging==24.2\npaddleocr==2.9.1\npaddlepaddle==3.0.0b2\npandas==2.2.3\npillow==11.0.0\nPPOCRLabel==2.1.11\npremailer==3.10.0\nprotobuf==5.28.3\npyclipper==1.3.0.post6\npydantic==2.9.2\npydantic_core==2.23.4\npyparsing==3.2.0\nPyQt5==5.15.11\nPyQt5-Qt5==5.15.15\nPyQt5_sip==12.15.0\npython-dateutil==2.9.0.post0\npython-docx==1.1.2\npytz==2024.2\nPyYAML==6.0.2\nRapidFuzz==3.10.1\nrequests==2.32.3\nscikit-image==0.24.0\nscikit-learn==1.5.2\nscipy==1.14.1\nshapely==2.0.6\nsix==1.16.0\nsniffio==1.3.1\nsoupsieve==2.6\ntermcolor==2.5.0\nthreadpoolctl==3.5.0\ntifffile==2024.9.20\ntomli==2.1.0\ntqdm==4.67.0\ntyping_extensions==4.12.2\ntzdata==2024.2\nurllib3==2.2.3\n```\n使用的环境是deepin v23 社区版本，不知道是否跟本机或者和开发环境有关，我不太会搭这个配置环境，还望指出问题所在。",
        "state": "open",
        "user": "WilliamGrant",
        "closed_by": null,
        "created_at": "2024-11-12T09:37:36+00:00",
        "updated_at": "2024-11-20T07:29:19+00:00",
        "closed_at": null,
        "comments_count": [
            "haimat",
            "GreatV",
            "haimat",
            "WilliamGrant"
        ],
        "labels": []
    },
    {
        "error": "'NoneType' object has no attribute 'repository'",
        "issue_number": 111
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 109,
        "title": "划分数据集错误",
        "body": "![Image](https://github.com/user-attachments/assets/c45f7620-056a-4fd5-ac94-31393aa8c1f9)\n",
        "state": "closed",
        "user": "gopower",
        "closed_by": "GreatV",
        "created_at": "2024-11-12T09:39:36+00:00",
        "updated_at": "2025-05-24T16:30:50+00:00",
        "closed_at": "2025-05-24T16:30:50+00:00",
        "comments_count": [],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 110,
        "title": "PPOCRLabel== 2.1.3 在windows11下发生以下异常",
        "body": "python 3.8 \n\n1.1 安装PaddlePaddle\n paddlepaddle-gpu==3.0.0b2\n\n1.2.1 通过whl包安装与运行\nWindows\npip install PPOCRLabel  # 安装\n\n# 选择标签模式来启动\nPPOCRLabel --lang ch  # 启动【普通模式】，用于打【检测+识别】场景的标签\n\n报错如下：\n\nTraceback (most recent call last):\n  File \"C:\\ProgramData\\miniconda3\\envs\\ppocrlabel\\lib\\runpy.py\", line 194, in _run_module_as_main\n    return _run_code(code, main_globals, None,\n  File \"C:\\ProgramData\\miniconda3\\envs\\ppocrlabel\\lib\\runpy.py\", line 87, in _run_code\n    exec(code, run_globals)\n  File \"C:\\ProgramData\\miniconda3\\envs\\ppocrlabel\\Scripts\\PPOCRLabel.exe\\__main__.py\", line 7, in <module>\n    sys.exit(main())\n  File \"C:\\ProgramData\\miniconda3\\envs\\ppocrlabel\\lib\\site-packages\\PPOCRLabel\\PPOCRLabel.py\", line 2828, in main\n    app, _win = get_main_app(sys.argv)\n  File \"C:\\ProgramData\\miniconda3\\envs\\ppocrlabel\\lib\\site-packages\\PPOCRLabel\\PPOCRLabel.py\", line 2818, in get_main_app\n    win = MainWindow(lang=args.lang,\n  File \"C:\\ProgramData\\miniconda3\\envs\\ppocrlabel\\lib\\site-packages\\PPOCRLabel\\PPOCRLabel.py\", line 112, in __init__\n    result = self.table_ocr('./data/paddle.png', return_ocr_result_in_table=True)\n  File \"C:\\ProgramData\\miniconda3\\envs\\ppocrlabel\\lib\\site-packages\\paddleocr\\paddleocr.py\", line 926, in __call__\n    res, _ = super().__call__(img, return_ocr_result_in_table, img_idx=img_idx)\n  File \"C:\\ProgramData\\miniconda3\\envs\\ppocrlabel\\lib\\site-packages\\PaddleOCR\\ppstructure\\predict_system.py\", line 163, in __call__\n    res, table_time_dict = self.table_system(\n  File \"C:\\ProgramData\\miniconda3\\envs\\ppocrlabel\\lib\\site-packages\\PaddleOCR\\ppstructure\\table\\predict_table.py\", line 96, in __call__\n    structure_res, elapse = self._structure(copy.deepcopy(img))\n  File \"C:\\ProgramData\\miniconda3\\envs\\ppocrlabel\\lib\\site-packages\\PaddleOCR\\ppstructure\\table\\predict_table.py\", line 118, in _structure\n    structure_res, elapse = self.table_structurer(copy.deepcopy(img))\n  File \"C:\\ProgramData\\miniconda3\\envs\\ppocrlabel\\lib\\site-packages\\PaddleOCR\\ppstructure\\table\\predict_structure.py\", line 140, in __call__\n    self.predictor.run()\nValueError: In user code:\n    InvalidArgumentError: The shape of input[0] and input[1] is expected to be equal.But received input[0]'s shape = [], input[1]'s shape = [1].\n      [Hint: Expected inputs_dims[i].size() == out_dims.size(), but received inputs_dims[i].size():1 != out_dims.size():0.] (at C:\\home\\workspace\\Paddle\\paddle/phi/kernels/funcs/concat_funcs.h:55)\n      [operator < pd_kernel.phi_kernel > error]\n\n\n请帮忙指导安装环境，谢谢\n",
        "state": "open",
        "user": "UWPStone",
        "closed_by": null,
        "created_at": "2024-11-12T14:54:19+00:00",
        "updated_at": "2024-11-13T03:10:53+00:00",
        "closed_at": null,
        "comments_count": [
            "GreatV",
            "UWPStone",
            "UWPStone",
            "GreatV",
            "GreatV"
        ],
        "labels": []
    },
    {
        "error": "'NoneType' object has no attribute 'repository'",
        "issue_number": 125
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 113,
        "title": "导出表格标注报错",
        "body": "\n![Image](https://github.com/user-attachments/assets/f58dbe7f-cdff-4bed-8c79-773f085fd476)\n\nTraceback (most recent call last):\n  File \"D:\\project\\PPOCRLabel-main\\PPOCRLabel.py\", line 3220, in exportJSON\n    token_list = convert_token(html_list)\n  File \"D:\\project\\PPOCRLabel-main\\libs\\utils.py\", line 238, in convert_token\n    token_list.append(' colspan=\"{}\"'.format(int(n)))\nValueError: invalid literal for int() with base 10: '3 rowspan=2'\n",
        "state": "open",
        "user": "qaz147okm",
        "closed_by": null,
        "created_at": "2024-11-21T05:43:53+00:00",
        "updated_at": "2024-11-21T05:43:53+00:00",
        "closed_at": null,
        "comments_count": [],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 112,
        "title": "PPOCRLabel自动标注没有标注",
        "body": "(ppocr_gpu) D:\\Anaconda\\envs\\ppocr_gpu\\Lib\\site-packages\\PPOCRLabel>python PPOCRLabel.py --lang ch\n信息: 用提供的模式无法找到文件。\nD:\\Anaconda\\envs\\ppocr_gpu\\lib\\site-packages\\paddle\\utils\\cpp_extension\\extension_utils.py:686: UserWarning: No ccache found. Please be aware that recompiling all source files may be required. You can download and install ccache from: https://github.com/ccache/ccache/blob/master/doc/INSTALL.md\n  warnings.warn(warning_message)\n[2024/11/18 10:41:33] ppocr WARNING: The first GPU is used for inference by default, GPU ID: 0\n[2024/11/18 10:41:34] ppocr WARNING: The first GPU is used for inference by default, GPU ID: 0\n[2024/11/18 10:41:35] ppocr WARNING: The first GPU is used for inference by default, GPU ID: 0\n[2024/11/18 10:41:37] ppocr WARNING: When args.layout is false, args.ocr is automatically set to false\n[2024/11/18 10:41:37] ppocr WARNING: The first GPU is used for inference by default, GPU ID: 0\n[2024/11/18 10:41:38] ppocr WARNING: The first GPU is used for inference by default, GPU ID: 0\n[2024/11/18 10:41:39] ppocr WARNING: The first GPU is used for inference by default, GPU ID: 0\n[2024/11/18 10:41:43] ppocr WARNING: The first GPU is used for inference by default, GPU ID: 0\n\n执行第一行的代码后弹出如上信息，然后打开了标注工具，但是点击“自动标注”后点击“OK”后，图片上没有任何标注。\n在执行自动标注前终端会弹出来这个Using model from  paddle，不知道有没有关系.....",
        "state": "closed",
        "user": "py0student",
        "closed_by": "py0student",
        "created_at": "2024-11-18T03:03:27+00:00",
        "updated_at": "2024-11-18T03:34:34+00:00",
        "closed_at": "2024-11-18T03:34:34+00:00",
        "comments_count": [
            "GreatV",
            "py0student"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 116,
        "title": "ppocrlabel表格标注合并cell的时候，源码中是根据生成的excel文件进行表格单元格合并代码的创建，那岂不是永远也标注不正确？",
        "body": "ppocrlabel表格标注合并cell的时候，源码中是根据自动生成的excel文件进行表格单元格合并代码的创建，我标注数据是因为已经认为表格自动标注是错误的，此时还使用自动生成的的cell合并，岂不是永远也标注不正确？\n![Image](https://github.com/user-attachments/assets/2346caa2-4ea7-4066-8935-d62bcdf15d74)\n",
        "state": "open",
        "user": "HUPg-95",
        "closed_by": null,
        "created_at": "2024-11-22T06:46:36+00:00",
        "updated_at": "2024-11-26T09:16:35+00:00",
        "closed_at": null,
        "comments_count": [
            "GreatV",
            "GreatV",
            "BotAndyGao"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 117,
        "title": "识别结果这里的文字必须是正确的吗？",
        "body": "<!-- Failed to upload \"image.png\" -->\n识别结果这里的文字必须是正确的吗？\n只训练表格结构，是不是只用标出文字框，不一定文字要正确。\n",
        "state": "open",
        "user": "carlos-yuan",
        "closed_by": null,
        "created_at": "2024-11-26T07:07:22+00:00",
        "updated_at": "2024-11-26T07:32:10+00:00",
        "closed_at": null,
        "comments_count": [
            "GreatV"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 122,
        "title": "自动标注会将已经确认好的图片重新执行标注",
        "body": "版本：2.1.11\n问题：当一个标注文件夹新增一张图片后，点击自动标注，会将其他已经确认保存好标注结果的图片，重新进行自动标注（文件前已经显示✅）\n",
        "state": "open",
        "user": "759325100",
        "closed_by": null,
        "created_at": "2024-12-03T09:57:51+00:00",
        "updated_at": "2024-12-04T03:25:59+00:00",
        "closed_at": null,
        "comments_count": [
            "759325100",
            "GreatV",
            "GreatV"
        ],
        "labels": []
    },
    {
        "error": "'NoneType' object has no attribute 'repository'",
        "issue_number": 126
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 118,
        "title": "标注表格数据生成的gt.txt无法通过PaddleX 表格训练的验证",
        "body": "生成的gt.txt跟原来的表格不一样\n原来的表格：\n![Image](https://github.com/user-attachments/assets/0c69059a-26cf-4ff1-8a20-e6f29c010ce6)\n\ngt.txt中gt属性 加线框后的html：\n![Image](https://github.com/user-attachments/assets/739f2714-d151-4aba-bacf-faf25a85969f)\n\ngt.txt （为了满足训练格式，手动加了‘images/’表示路径）：\n[gt.txt](https://github.com/user-attachments/files/17915056/gt.txt)\n\n验证数据集：\n![Image](https://github.com/user-attachments/assets/e0496623-ba6e-4dd7-bad0-a82efa9ae824)\n\n验证数据集命令：\npython main.py -c paddlex/configs/table_recognition/SLANet_plus.yaml \\\n    -o Global.mode=check_dataset \\\n    -o Global.dataset_dir=/home/aistudio/PaddleX/train_data\n\n验证失败出现的错误：\naistudio@jupyter-13749311-8523997:~/PaddleX$ python main.py -c paddlex/configs/table_recognition/SLANet_plus.yaml     -o Global.mode=check_dataset     -o Global.dataset_dir=/home/aistudio/PaddleX/train_data\n        Traceback (most recent call last):\n  File \"/home/aistudio/PaddleX/paddlex/utils/result_saver.py\", line 29, in wrap\n    result = func(self, *args, **kwargs)\n  File \"/home/aistudio/PaddleX/paddlex/engine.py\", line 38, in run\n    return self._model.check_dataset()\n  File \"/home/aistudio/PaddleX/paddlex/model.py\", line 90, in check_dataset\n    return dataset_checker.check()\n  File \"/home/aistudio/PaddleX/paddlex/modules/base/dataset_checker/dataset_checker.py\", line 75, in check\n    attrs = self.check_dataset(dataset_dir)\n  File \"/home/aistudio/PaddleX/paddlex/modules/table_recognition/dataset_checker/__init__.py\", line 67, in check_dataset\n    return check(dataset_dir, self.global_config.output, sample_num=10)\n  File \"/home/aistudio/PaddleX/paddlex/modules/table_recognition/dataset_checker/dataset_src/check_dataset.py\", line 75, in check\n    raise CheckFailedError(\npaddlex.utils.errors.dataset_checker.CheckFailedError: Check dataset failed. We encountered the following error:\n  The number of cells needs to be consistent with the number of tokens but the number of cells is {boxes_num}, and the number of tokens is {tokens_num}.",
        "state": "closed",
        "user": "zx007nice",
        "closed_by": "GreatV",
        "created_at": "2024-11-26T08:00:42+00:00",
        "updated_at": "2024-11-28T08:42:45+00:00",
        "closed_at": "2024-11-28T08:42:45+00:00",
        "comments_count": [
            "GreatV",
            "zx007nice",
            "GreatV",
            "zx007nice",
            "GreatV",
            "zx007nice",
            "GreatV",
            "zx007nice",
            "GreatV",
            "zx007nice",
            "zx007nice",
            "BotAndyGao",
            "BotAndyGao",
            "zx007nice",
            "BotAndyGao",
            "zx007nice",
            "BotAndyGao",
            "carlos-yuan",
            "carlos-yuan",
            "BotAndyGao",
            "zx007nice"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 124,
        "title": "Traceback (most recent call last):   File \"PPOCRLabel.py\", line 80, in <module>     from paddleocr import PaddleOCR, PPStructure ImportError: cannot import name 'PaddleOCR' from 'paddleocr' (unknown location)",
        "body": "Traceback (most recent call last):\n  File \"PPOCRLabel.py\", line 80, in <module>\n    from paddleocr import PaddleOCR, PPStructure\nImportError: cannot import name 'PaddleOCR' from 'paddleocr' (unknown location)",
        "state": "open",
        "user": "nissansz",
        "closed_by": null,
        "created_at": "2024-12-16T04:33:01+00:00",
        "updated_at": "2024-12-16T04:58:16+00:00",
        "closed_at": null,
        "comments_count": [
            "GreatV",
            "nissansz"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 129,
        "title": "使用python脚本运行，报错如下。使用其他方式运行，除了新窗口出现一会儿再闪退外，没有其他反应",
        "body": "这是python脚本运行（python PPOCRLabel.py --lang ch）的报错信息：\n信息: 用提供的模式无法找到文件。\nD:\\a\\pythonTool\\Anaconda3\\envs\\paddleocr\\lib\\site-packages\\paddle\\utils\\cpp_extension\\extension_utils.py:686: UserWarning: No ccache found. Please be aware that recompiling all source files may be required. You can download and install ccache from: https://github.com/ccache/ccache/blob/master/doc/INSTALL.md\n  warnings.warn(warning_message)\n[2024/12/25 19:39:19] ppocr WARNING: When args.layout is false, args.ocr is automatically set to false\nTraceback (most recent call last):\n  File \"D:\\a\\paddlepaddle\\PPOCRLabel\\PPOCRLabel.py\", line 3667, in <module>\n    sys.exit(main())\n  File \"D:\\a\\paddlepaddle\\PPOCRLabel\\PPOCRLabel.py\", line 3662, in main\n    app, _win = get_main_app(sys.argv)\n  File \"D:\\a\\paddlepaddle\\PPOCRLabel\\PPOCRLabel.py\", line 3642, in get_main_app\n    win = MainWindow(\n  File \"D:\\a\\paddlepaddle\\PPOCRLabel\\PPOCRLabel.py\", line 213, in __init__\n    result = self.table_ocr(\n  File \"D:\\a\\pythonTool\\Anaconda3\\envs\\paddleocr\\lib\\site-packages\\paddleocr\\paddleocr.py\", line 926, in __call__\n    res, _ = super().__call__(img, return_ocr_result_in_table, img_idx=img_idx)\n  File \"D:\\a\\pythonTool\\Anaconda3\\envs\\paddleocr\\lib\\site-packages\\paddleocr\\ppstructure\\predict_system.py\", line 163, in __call__\n    res, table_time_dict = self.table_system(\n  File \"D:\\a\\pythonTool\\Anaconda3\\envs\\paddleocr\\lib\\site-packages\\paddleocr\\ppstructure\\table\\predict_table.py\", line 96, in __call__\n    structure_res, elapse = self._structure(copy.deepcopy(img))\n  File \"D:\\a\\pythonTool\\Anaconda3\\envs\\paddleocr\\lib\\site-packages\\paddleocr\\ppstructure\\table\\predict_table.py\", line 118, in _structure\n    structure_res, elapse = self.table_structurer(copy.deepcopy(img))\n  File \"D:\\a\\pythonTool\\Anaconda3\\envs\\paddleocr\\lib\\site-packages\\paddleocr\\ppstructure\\table\\predict_structure.py\", line 140, in __call__\n    self.predictor.run()\nValueError: In user code:\n\n\n    InvalidArgumentError: The shape of input[0] and input[1] is expected to be equal.But received input[0]'s shape = [], input[1]'s shape = [1].\n      [Hint: Expected inputs_dims[i].size() == out_dims.size(), but received inputs_dims[i].size():1 != out_dims.size():0.] (at ..\\paddle/phi/kernels/funcs/concat_funcs.h:55)\n      [operator < pd_kernel.phi_kernel > error]\n\nconda新环境下，python 3.10，paddlepaddle已装，一切要求都仔细检查过\n这是怎么回事呢？期望有大佬指教下",
        "state": "open",
        "user": "tlishi",
        "closed_by": null,
        "created_at": "2024-12-25T11:42:40+00:00",
        "updated_at": "2024-12-26T03:43:58+00:00",
        "closed_at": null,
        "comments_count": [
            "GreatV",
            "tlishi"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 127,
        "title": "PPOCRLabel没看到关键信息提取怎样标注",
        "body": "我是用PPOCRLabel工具时，没看到怎样标注关键信息，？",
        "state": "open",
        "user": "jiangming7301",
        "closed_by": null,
        "created_at": "2024-12-16T09:43:55+00:00",
        "updated_at": "2024-12-16T09:48:02+00:00",
        "closed_at": null,
        "comments_count": [
            "GreatV"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 130,
        "title": "为什么新版本中c/x快捷键旋转标注框的功能取消了？",
        "body": "为什么新版本中c/x快捷键旋转标注框的功能取消了，这个功能明明比z/x/c/v选择四点分别调整更加便捷。如果我想使用新版本的其他新功能，有想使用c/x选择标注框，该在哪里修改代码实现",
        "state": "open",
        "user": "gokamisama",
        "closed_by": null,
        "created_at": "2024-12-27T07:25:45+00:00",
        "updated_at": "2025-06-14T00:14:20+00:00",
        "closed_at": null,
        "comments_count": [],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 128,
        "title": "表格标注后生成的html有问题",
        "body": "表格标注之后生成的gt.txt中的html格式不正确，检查标注和excel都没有问题，就是最后的html格式不正确，重新点表格识别，重新标注后结果正确。标注500张有50张不正确的",
        "state": "open",
        "user": "linqi1321",
        "closed_by": null,
        "created_at": "2024-12-22T15:28:46+00:00",
        "updated_at": "2024-12-22T15:36:00+00:00",
        "closed_at": null,
        "comments_count": [
            "GreatV"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 131,
        "title": "表格内部单元格存在换行如何标注",
        "body": "这种表格该如何标注",
        "state": "open",
        "user": "zhangxj-1",
        "closed_by": null,
        "created_at": "2025-01-02T06:26:58+00:00",
        "updated_at": "2025-01-15T10:54:39+00:00",
        "closed_at": null,
        "comments_count": [
            "GreatV",
            "GreatV",
            "zhangxj-1",
            "GreatV",
            "zhangxj-1",
            "GreatV",
            "GreatV",
            "zhangxj-1",
            "zhangxj-1",
            "GreatV"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 133,
        "title": "Linux 下无法通过fcitx4/5输入中文",
        "body": "环境\n```\nManjaro Linux x86_64\nKernel: Linux 5.15.173-1-MANJARO\nconda: \n  python==3.10.16\n  paddleocr                 2.9.1                    pypi_0    pypi\n  paddlepaddle-cuda         12.3                 many_linux    paddle\n  paddlepaddle-gpu          3.0.0rc1        py310_gpu_cuda12.3_many_linux    paddle\n\n```\nfcitx5 与fcitx4均测试过，无法输入中文。fcitx已按照 [ArchWiki-Fcitx5](https://wiki.archlinuxcn.org/wiki/Fcitx5) 设置配置：\n```\n#/etc/environment\nGTK_IM_MODULE=fcitx\nQT_IM_MODULE=fcitx\nXMODIFIERS=@im=fcitx\nSDL_IM_MODULE=fcitx\nGLFW_IM_MODULE=ibus\n\n```",
        "state": "open",
        "user": "jiangjiechu",
        "closed_by": null,
        "created_at": "2025-02-04T11:21:03+00:00",
        "updated_at": "2025-02-04T13:00:37+00:00",
        "closed_at": null,
        "comments_count": [
            "GreatV",
            "jiangjiechu"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 132,
        "title": "关键信息提取模型标注和训练支持几种标签",
        "body": "看到XFUND数据集支持“header question answer other”四种标签，自定义数据集支持除此四种以外自定义的其他种类的标签吗？",
        "state": "open",
        "user": "zhangxj-1",
        "closed_by": null,
        "created_at": "2025-01-08T08:08:02+00:00",
        "updated_at": "2025-01-08T08:14:23+00:00",
        "closed_at": null,
        "comments_count": [
            "GreatV"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 138,
        "title": "按照readme安装后报错RuntimeError: (PreconditionNotMet) Tensor's dimension is out of bound",
        "body": "我在我自己公司的电脑安装，都不会报错，但是在家里和其他同事的电脑就是无法安装，\nTraceback (most recent call last):\n  File \"C:\\Users\\bob\\Desktop\\PPOCRLabe\\PPOCRLabel.py\", line 3698, in <module>\n    sys.exit(main())\n  File \"C:\\Users\\bob\\Desktop\\PPOCRLabe\\PPOCRLabel.py\", line 3693, in main\n    app, _win = get_main_app(sys.argv)\n  File \"C:\\Users\\bob\\Desktop\\PPOCRLabe\\PPOCRLabel.py\", line 3673, in get_main_app\n    win = MainWindow(\n  File \"C:\\Users\\bob\\Desktop\\PPOCRLabe\\PPOCRLabel.py\", line 208, in __init__\n    self.table_ocr = PPStructure(\n  File \"D:\\soft-install\\anaconda3\\envs\\paddle_env\\lib\\site-packages\\paddleocr\\paddleocr.py\", line 900, in __init__\n    super().__init__(params)\n  File \"D:\\soft-install\\anaconda3\\envs\\paddle_env\\lib\\site-packages\\paddleocr\\ppstructure\\predict_system.py\", line 82, in __init__\n    self.table_system = TableSystem(args)\n  File \"D:\\soft-install\\anaconda3\\envs\\paddle_env\\lib\\site-packages\\paddleocr\\ppstructure\\table\\predict_table.py\", line 79, in __init__\n    self.table_structurer = predict_strture.TableStructurer(args)\n  File \"D:\\soft-install\\anaconda3\\envs\\paddle_env\\lib\\site-packages\\paddleocr\\ppstructure\\table\\predict_structure.py\", line 97, in __init__\n    ) = utility.create_predictor(args, \"table\", logger)\n  File \"D:\\soft-install\\anaconda3\\envs\\paddle_env\\lib\\site-packages\\paddleocr\\tools\\infer\\utility.py\", line 364, in create_predictor\n    predictor = inference.create_predictor(config)\nRuntimeError: (PreconditionNotMet) Tensor's dimension is out of bound.Tensor's dimension must be equal or less than the size of its memory.But received Tensor's dimension is 8, memory's size is 0.\n  [Hint: Expected numel() * SizeOf(dtype()) <= memory_size(), but received numel() * SizeOf(dtype()):8 > memory_size():0.] (at ..\\paddle\\phi\\core\\dense_tensor_impl.cc:55)\n  [operator < scale > error]\n\n貌似不是paddlepaddle或者paddleocr之类的版本问题，看了issues也没有找到遇到相似问题",
        "state": "open",
        "user": "TheHonestBob",
        "closed_by": null,
        "created_at": "2025-03-17T09:41:22+00:00",
        "updated_at": "2025-03-24T14:03:03+00:00",
        "closed_at": null,
        "comments_count": [
            "GreatV",
            "GreatV",
            "TheHonestBob",
            "GreatV",
            "TheHonestBob",
            "owmo-2000"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 144,
        "title": "大佬们，按照文档装的，但是RuntimeError: (PreconditionNotMet) Tensor's dimension is out of bound.Tensor's dimension must be equal or less than the size of its memory.But received Tensor's dimension is 8, memory's size is 0.",
        "body": "![Image](https://github.com/user-attachments/assets/a2741f14-7dc9-4464-be53-e51c7aca526b)\n这是运行PPOCRLabel --lang ch的报错信息。\n\n![Image](https://github.com/user-attachments/assets/cef76a3e-343e-494a-9a9f-77d3b6324dce)\n这是paddlepaddle，和cuda的版本。\n我的电脑支持12.0，但是在该环境下安装的时cuda11.8。其实我也试过12.0的，也存在错误。\n12.0的cuda在运行PPOCRLabel --lang ch时，鼠标指针会变成等待的状态，但是过个1秒中就停了，然后终端直接回到了输入命令的状态，相应的窗口没有打开。\n\n大佬能帮我看看吗？我已经装了两天了，人都要晕了。",
        "state": "open",
        "user": "owmo-2000",
        "closed_by": null,
        "created_at": "2025-03-24T14:32:07+00:00",
        "updated_at": "2025-03-26T06:33:36+00:00",
        "closed_at": null,
        "comments_count": [
            "GreatV",
            "GreatV",
            "zzg4291105",
            "owmo-2000",
            "owmo-2000",
            "GreatV",
            "yangwenhao"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 145,
        "title": "PPOCRLabel-v2.2.1.exe自动标注中途会闪退",
        "body": "PPOCRLabel-v2.2.1.exe，执行自动标注中途会闪退。",
        "state": "open",
        "user": "yangwenhao",
        "closed_by": null,
        "created_at": "2025-03-26T06:58:50+00:00",
        "updated_at": "2025-03-27T03:04:06+00:00",
        "closed_at": null,
        "comments_count": [
            "GreatV",
            "GreatV",
            "yangwenhao",
            "yangwenhao",
            "yangwenhao",
            "GreatV"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 146,
        "title": "表格识别时，单元格重识别会闪退",
        "body": "点击表格识别后发现有空单元格，框选空单元格，右键，选择单元格重识别会导致PPOCRLabel闪退，报错信息如下：\n`Traceback (most recent call last):\n  File \"C:\\Users\\iamzh\\PycharmProjects\\PPOCRLabel\\PPOCRLabel.py\", line 3182, in cellreRecognition\n    if len(bboxes) > 0:\nTypeError: object of type 'NoneType' has no len()`\npython版本：3.10.9\npaddlepaddle-gpu版本：3.0.0rc1\npaddleocr版本：2.10.0",
        "state": "open",
        "user": "Ha11oWe1t",
        "closed_by": null,
        "created_at": "2025-04-09T06:35:37+00:00",
        "updated_at": "2025-06-16T04:22:57+00:00",
        "closed_at": null,
        "comments_count": [
            "GreatV",
            "Sam-gsj"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 148,
        "title": "MacBook M3 MacOS15.4上有时出错",
        "body": "版本：\nPython 3.13.3\npaddleocr                      2.10.0\npaddlepaddle                   3.0.0\nPPOCRLabel                     2.2.1\n运行PPOCRLabel --lang ch，自动标注遇到特定图片总会出错，终端输出：\n(NotFound) No allocator found for the place, Place(undefined:0)\n  [Hint: Expected iter != allocators.end(), but received iter == allocators.end().] (at /Users/paddle/xly/workspace/7b5bf7ed-f223-4c37-9311-a1c9d604457b/Paddle/paddle/phi/core/memory/allocation/allocator_facade.cc:381)\n  [operator < matmul > error]\nTraceback (most recent call last):\n  File \"/Users/usrname/ocrpath/ppocr-venv/lib/python3.13/site-packages/PPOCRLabel/libs/autoDialog.py\", line 45, in run\n    self.result_dic = self.ocr.ocr(Imgpath, cls=True, det=True)[\n                      ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/usrname/ocrpath/ppocr-venv/lib/python3.13/site-packages/paddleocr/paddleocr.py\", line 766, in ocr\n    dt_boxes, rec_res, _ = self.__call__(img, cls, slice)\n                           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/Users/usrname/ocrpath/ppocr-venv/lib/python3.13/site-packages/paddleocr/tools/infer/predict_system.py\", line 144, in __call__\n    rec_res, elapse = self.text_recognizer(img_crop_list)\n                      ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^\n  File \"/Users/usrname/ocrpath/ppocr-venv/lib/python3.13/site-packages/paddleocr/tools/infer/predict_rec.py\", line 793, in __call__\n    self.predictor.run()\n    ~~~~~~~~~~~~~~~~~~^^\nRuntimeError: (NotFound) No allocator found for the place, Place(undefined:0)\n  [Hint: Expected iter != allocators.end(), but received iter == allocators.end().] (at /Users/paddle/xly/workspace/7b5bf7ed-f223-4c37-9311-a1c9d604457b/Paddle/paddle/phi/core/memory/allocation/allocator_facade.cc:381)\n  [operator < matmul > error]\n确认以跳过相应图片即可顺利进行，且有时使用表格识别时也会出错：\nTraceback (most recent call last):\n  File \"/Users/usrname/ocrpath/ppocr-venv/lib/python3.13/site-packages/PPOCRLabel/PPOCRLabel.py\", line 3028, in TableRecognition\n    res = self.table_ocr(img, return_ocr_result_in_table=True)\n  File \"/Users/usrname/ocrpath/ppocr-venv/lib/python3.13/site-packages/paddleocr/paddleocr.py\", line 932, in __call__\n    res, _ = super().__call__(img, return_ocr_result_in_table, img_idx=img_idx)\n             ~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/usrname/ocrpath/ppocr-venv/lib/python3.13/site-packages/paddleocr/ppstructure/predict_system.py\", line 163, in __call__\n    res, table_time_dict = self.table_system(\n                           ~~~~~~~~~~~~~~~~~^\n        roi_img, return_ocr_result_in_table\n        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n    )\n    ^\n  File \"/Users/usrname/ocrpath/ppocr-venv/lib/python3.13/site-packages/paddleocr/ppstructure/table/predict_table.py\", line 100, in __call__\n    dt_boxes, rec_res, det_elapse, rec_elapse = self._ocr(copy.deepcopy(img))\n                                                ~~~~~~~~~^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/usrname/ocrpath/ppocr-venv/lib/python3.13/site-packages/paddleocr/ppstructure/table/predict_table.py\", line 145, in _ocr\n    rec_res, rec_elapse = self.text_recognizer(img_crop_list)\n                          ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^\n  File \"/Users/usrname/ocrpath/ppocr-venv/lib/python3.13/site-packages/paddleocr/tools/infer/predict_rec.py\", line 793, in __call__\n    self.predictor.run()\n    ~~~~~~~~~~~~~~~~~~^^\nRuntimeError: (NotFound) No allocator found for the place, Place(undefined:0)\n  [Hint: Expected iter != allocators.end(), but received iter == allocators.end().] (at /Users/paddle/xly/workspace/7b5bf7ed-f223-4c37-9311-a1c9d604457b/Paddle/paddle/phi/core/memory/allocation/allocator_facade.cc:381)\n  [operator < matmul > error]\n有时使用多点标注识别时也会出错（同样有RuntimeError: (NotFound) No allocator found for the place, Place(undefined:0)，详细输出略）",
        "state": "closed",
        "user": "sunxis",
        "closed_by": "sunxis",
        "created_at": "2025-04-14T05:39:16+00:00",
        "updated_at": "2025-05-14T05:35:57+00:00",
        "closed_at": "2025-04-18T12:17:38+00:00",
        "comments_count": [
            "GreatV",
            "GreatV",
            "sunxis",
            "GreatV",
            "kaiwensun",
            "sunxis",
            "alexChanCN"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 147,
        "title": "Export Recognition Result is Rotated",
        "body": "PPOCRLabel v2.2.1 (https://pypi.org/project/PPOCRLabel/)\nAfter labelling an image and exporting the recognition result with: file -> Export Recognition Result\nThe output image is rotated 90 degrees.\n\nInput labelled image\n\n![Image](https://github.com/user-attachments/assets/6f0f898c-26cf-4729-b567-ccadfb37853c)\n\nOutput\n\n![Image](https://github.com/user-attachments/assets/214c61f6-d50c-466b-91bd-90781b6f87f7)\n\nI'm not sure why the exported recognition result is rotated from the original. Is there anyway to fix this?",
        "state": "closed",
        "user": "ConnorMcGuirk",
        "closed_by": "ConnorMcGuirk",
        "created_at": "2025-04-11T19:48:36+00:00",
        "updated_at": "2025-05-02T16:42:29+00:00",
        "closed_at": "2025-05-02T16:42:29+00:00",
        "comments_count": [
            "GreatV",
            "ConnorMcGuirk",
            "ConnorMcGuirk",
            "LovingThresh",
            "LovingThresh",
            "ConnorMcGuirk"
        ],
        "labels": [
            "bug"
        ]
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 150,
        "title": "这种不规则结构的表格，应该如何标注？",
        "body": "表格结构识别任务，在制作微调的数据集时，表格数据如果如下图这种，接待标准这有5列，下面的接待类型有3列\n\n![Image](https://github.com/user-attachments/assets/505885b0-7aad-48f7-81a3-fbbfad81e86f)\n\n这样在excel里的表格结构就会宽度不一致\n\n![Image](https://github.com/user-attachments/assets/2187ef8a-f2c4-474b-bbc0-4f32a300f2c4)\n\n是否应该将接待标准的5列合并为一列，接待类型的3列也合并为一列？还是按照上图这样就行？",
        "state": "open",
        "user": "Surrin1999",
        "closed_by": null,
        "created_at": "2025-04-16T02:54:07+00:00",
        "updated_at": "2025-04-16T03:37:52+00:00",
        "closed_at": null,
        "comments_count": [
            "GreatV",
            "Surrin1999",
            "Surrin1999"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 149,
        "title": "微调表格结构识别模型在标注数据时，右边的识别结果是否要准确？",
        "body": "是不是只要保证顺序正确（从左到右、从上到下）就行，这里的文字不一定要准确？\n![Image](https://github.com/user-attachments/assets/81ad8f51-4ebd-45f5-bbe2-3a000cb0f2b9)",
        "state": "open",
        "user": "Surrin1999",
        "closed_by": null,
        "created_at": "2025-04-16T02:31:55+00:00",
        "updated_at": "2025-06-12T14:48:19+00:00",
        "closed_at": null,
        "comments_count": [
            "GreatV",
            "Surrin1999",
            "Sam-gsj"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 151,
        "title": "很多问题",
        "body": "信息里面app version是1.0.0，是最新版吗？然后readme里面提到的自动重新识别和自动打勾的功能都没有，表格识别倒是有\n自动标注使用的时候对小尺寸图片不进行识别，但是手动标注个矩形框后重新识别又能识别了，那为什么要不识别小尺寸图片\n对识别结果修改时，无法输入中文，输入法设置为中文，输入第一个字母后就强制变成英文",
        "state": "open",
        "user": "ignorance-hpf",
        "closed_by": null,
        "created_at": "2025-05-07T12:20:57+00:00",
        "updated_at": "2025-06-13T03:57:16+00:00",
        "closed_at": null,
        "comments_count": [
            "Sam-gsj"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 152,
        "title": "使用旋转功能时程序崩溃",
        "body": "错误信息\nTraceback (most recent call last):\n  File \"D:\\anaconda3\\envs\\paddle_env\\lib\\site-packages\\PPOCRLabel\\PPOCRLabel.py\", line 897, in rotateImgAction\n    self.rotateImg(filename=filename, k=k, _value=True)\n  File \"D:\\anaconda3\\envs\\paddle_env\\lib\\site-packages\\PPOCRLabel\\PPOCRLabel.py\", line 874, in rotateImg\n    pix = np.rot90(pix, k)\n  File \"<__array_function__ internals>\", line 200, in rot90\n  File \"D:\\anaconda3\\envs\\paddle_env\\lib\\site-packages\\numpy\\lib\\function_base.py\", line 227, in rot90\n    raise ValueError(\"Axes={} out of range for array of ndim={}.\"\nValueError: Axes=(0, 1) out of range for array of ndim=0.\n\n删除图像需要安装win32依赖",
        "state": "open",
        "user": "xianSir",
        "closed_by": null,
        "created_at": "2025-05-09T09:41:37+00:00",
        "updated_at": "2025-06-13T04:01:17+00:00",
        "closed_at": null,
        "comments_count": [
            "Sam-gsj"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 153,
        "title": "运行时报错",
        "body": "(ppocrlabel) PS E:\\PPOCRLabel-main> python PPOCRLabel.py --lang ch                                                                                  \n[2025/05/21 16:59:22] ppocr WARNING: When args.layout is false, args.ocr is automatically set to false\nTraceback (most recent call last):\n  File \"PPOCRLabel.py\", line 3698, in <module>\n    sys.exit(main())\n  File \"PPOCRLabel.py\", line 3693, in main\n    app, _win = get_main_app(sys.argv)\n  File \"PPOCRLabel.py\", line 3673, in get_main_app\n    win = MainWindow(\n  File \"PPOCRLabel.py\", line 213, in __init__\n    result = self.ocr.ocr(\"./data/paddle.png\", cls=True, det=True)\n  File \"E:\\users\\KuangD\\AppData\\Local\\miniforge3\\envs\\ppocrlabel\\lib\\site-packages\\paddleocr\\paddleocr.py\", line 555, in ocr\n    dt_boxes, rec_res, _ = self.__call__(img, cls)\n  File \"E:\\users\\KuangD\\AppData\\Local\\miniforge3\\envs\\ppocrlabel\\lib\\site-packages\\paddleocr\\tools\\infer\\predict_system.py\", line 89, in __call__\n    img_crop_list, angle_list, elapse = self.text_classifier(\n  File \"E:\\users\\KuangD\\AppData\\Local\\miniforge3\\envs\\ppocrlabel\\lib\\site-packages\\paddleocr\\tools\\infer\\predict_cls.py\", line 114, in __call__     \n    cls_result = self.postprocess_op(prob_out)\n  File \"E:\\users\\KuangD\\AppData\\Local\\miniforge3\\envs\\ppocrlabel\\lib\\site-packages\\paddleocr\\ppocr\\postprocess\\cls_postprocess.py\", line 37, in __call__\n    decode_out = [(label_list[idx], preds[i, idx])\n  File \"E:\\users\\KuangD\\AppData\\Local\\miniforge3\\envs\\ppocrlabel\\lib\\site-packages\\paddleocr\\ppocr\\postprocess\\cls_postprocess.py\", line 37, in <listcomp>\n    decode_out = [(label_list[idx], preds[i, idx])\nTypeError: only integer scalar arrays can be converted to a scalar index\n运行时报以上错误，是代码逻辑问题吗\n",
        "state": "open",
        "user": "Dankuang-Dotcom",
        "closed_by": null,
        "created_at": "2025-05-21T09:52:41+00:00",
        "updated_at": "2025-05-22T07:26:12+00:00",
        "closed_at": null,
        "comments_count": [
            "TimouL",
            "Dankuang-Dotcom"
        ],
        "labels": []
    },
    {
        "error": "'NoneType' object has no attribute 'repository'",
        "issue_number": 162
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 155,
        "title": "绝了 配置了快一个星期了 各种问题",
        "body": "OMP: Error #15: Initializing libiomp5md.dll, but found libiomp5md.dll already initialized.\nOMP: Hint This means that multiple copies of the OpenMP runtime have been linked into the program. That is dangerous, since it can degrade performance or cause incorrect results. The best thing to do is to ensure that only a single OpenMP runtime is linked into the process, e.g. by avoiding static linking of the OpenMP runtime in any library. As an unsafe, unsupported, undocumented workaround you can set the environment variable KMP_DUPLICATE_LIB_OK=TRUE to allow the program to continue to execute, but that may cause crashes or silently produce incorrect results. For more information, please see http://www.intel.com/software/products/support/.",
        "state": "open",
        "user": "liujianghao1023",
        "closed_by": null,
        "created_at": "2025-05-24T16:20:32+00:00",
        "updated_at": "2025-06-06T12:02:10+00:00",
        "closed_at": null,
        "comments_count": [
            "GreatV",
            "liujianghao1023",
            "GreatV",
            "liujianghao1023",
            "liujianghao1023",
            "liujianghao1023",
            "GreatV",
            "serendipityshe",
            "GreatV"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 156,
        "title": "配置不出",
        "body": "绝了 真配置不出来 各种问题 一比一按照readme还是各种问题\n",
        "state": "open",
        "user": "liujianghao1023",
        "closed_by": null,
        "created_at": "2025-05-25T06:21:04+00:00",
        "updated_at": "2025-06-13T04:00:07+00:00",
        "closed_at": null,
        "comments_count": [
            "Sam-gsj"
        ],
        "labels": []
    },
    {
        "error": "'NoneType' object has no attribute 'repository'",
        "issue_number": 164
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 157,
        "title": "qiuzhu",
        "body": "   InvalidArgumentError: Input 'Mask' in SelectInputOp is invalid. 'Mask' must be less than the size of input vector 'X'. But received Mask = 1174289061, X's size = 2.\n      [Hint: Expected output_branch < x_names.size(), but received output_branch:1174289061 >= x_names.size():2.] (at ..\\paddle/fluid/operators/select_input_op.cc:51)\n      [operator < select_input > error]",
        "state": "open",
        "user": "liujianghao1023",
        "closed_by": null,
        "created_at": "2025-05-25T07:24:49+00:00",
        "updated_at": "2025-06-13T03:58:37+00:00",
        "closed_at": null,
        "comments_count": [
            "Sam-gsj"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 158,
        "title": "一堆问题，兼容性没搞好就不要放出来，百度搞这个的这帮人可以都裁了",
        "body": "兼容性搞的乱七八糟，mac下配置了几天都没搞好，各种坑\nPPOCRLabel --lang ch\nTraceback (most recent call last):\n  File \"/Library/Frameworks/Python.framework/Versions/3.11/bin/PPOCRLabel\", line 5, in <module>\n    from PPOCRLabel.PPOCRLabel import main\n  File \"/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/PPOCRLabel/PPOCRLabel.py\", line 81, in <module>\n    from paddleocr import PaddleOCR, PPStructure\nImportError: cannot import name 'PPStructure' from 'paddleocr' (/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/paddleocr/__init__.py)",
        "state": "closed",
        "user": "mowencode",
        "closed_by": "GreatV",
        "created_at": "2025-05-26T03:52:00+00:00",
        "updated_at": "2025-06-03T12:07:55+00:00",
        "closed_at": "2025-06-03T12:07:55+00:00",
        "comments_count": [
            "GreatV",
            "mowencode"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 159,
        "title": "release里面的mac版后缀是啥",
        "body": "怎么运行起来",
        "state": "closed",
        "user": "mowencode",
        "closed_by": "GreatV",
        "created_at": "2025-05-26T04:13:00+00:00",
        "updated_at": "2025-06-03T12:07:42+00:00",
        "closed_at": "2025-06-03T12:07:42+00:00",
        "comments_count": [],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 160,
        "title": "这个开源项目真的太恶心了",
        "body": null,
        "state": "closed",
        "user": "mowencode",
        "closed_by": "GreatV",
        "created_at": "2025-05-26T04:14:36+00:00",
        "updated_at": "2025-06-03T12:07:34+00:00",
        "closed_at": "2025-06-03T12:07:34+00:00",
        "comments_count": [],
        "labels": []
    },
    {
        "error": "'NoneType' object has no attribute 'repository'",
        "issue_number": 165
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 163,
        "title": "表格结构标注时，标注框经常出现坐标错误",
        "body": "![Image](https://github.com/user-attachments/assets/5e8ded78-eb1d-40a8-b3fa-190f5f819434)\n![Image](https://github.com/user-attachments/assets/a971064f-3426-4bbe-a150-7203d3c5567b)只有在确认然后返回查看后在能与坐标吻合，很多标好的图片返回过去看是不对齐的。",
        "state": "open",
        "user": "2Citiz",
        "closed_by": null,
        "created_at": "2025-06-04T05:56:20+00:00",
        "updated_at": "2025-06-04T08:23:19+00:00",
        "closed_at": null,
        "comments_count": [
            "GreatV",
            "2Citiz",
            "GreatV",
            "2Citiz"
        ],
        "labels": []
    },
    {
        "error": "'NoneType' object has no attribute 'repository'",
        "issue_number": 179
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 172,
        "title": "多表格标注",
        "body": "请问如果一张图片上有两个表格，应该怎么标注？我看自动的结果是在一个表格内，代码也是取的第一个sheet",
        "state": "open",
        "user": "fengerzou",
        "closed_by": null,
        "created_at": "2025-06-13T09:34:54+00:00",
        "updated_at": "2025-06-13T09:34:54+00:00",
        "closed_at": null,
        "comments_count": [],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 167,
        "title": "添加新的按钮",
        "body": "我想在界面中添加新的按钮，我在./resources/strings/*.properties均添加我的按钮信息，但是运行发现无法找到string id，我猜测是存在缓存文件，我该如何实现我的需求呢，以下是我的修改：\nstrings-en.properties\nresortposition=Resort position\nresortpositiondetail=Resort the bounding box positions\n\nstrings-zh-CN.properties\nresortposition=重新排序位置\nresortpositiondetail=重新排序坐标框位置",
        "state": "open",
        "user": "Sam-gsj",
        "closed_by": null,
        "created_at": "2025-06-11T12:21:20+00:00",
        "updated_at": "2025-06-11T12:52:24+00:00",
        "closed_at": null,
        "comments_count": [
            "GreatV",
            "Sam-gsj",
            "Sam-gsj",
            "GreatV",
            "Sam-gsj"
        ],
        "labels": []
    },
    {
        "error": "'NoneType' object has no attribute 'repository'",
        "issue_number": 183
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 168,
        "title": "无法启动",
        "body": "https://github.com/PFCCLab/PPOCRLabel/releases/tag/v3.0.1\n我使用此连接下载， 双击后报错如下\n\n![Image](https://github.com/user-attachments/assets/cfa22593-ae21-4c28-be78-60522967298e)\n\n\npip install 后 启动不报错  但是没有任何界面 也没有任何后台",
        "state": "open",
        "user": "strikene",
        "closed_by": null,
        "created_at": "2025-06-12T08:40:25+00:00",
        "updated_at": "2025-07-04T01:59:56+00:00",
        "closed_at": null,
        "comments_count": [
            "GreatV",
            "GreatV",
            "13691126988"
        ],
        "labels": [
            "bug"
        ]
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 169,
        "title": "20250613当前版本是默认不支持PP-OCRv4版本训练的模型?",
        "body": "\n指定自己训练的模型作为自动标注报错\n```bash\n    model_name == config[\"Global\"][\"model_name\"]\nAssertionError: Model name mismatch，please input the correct model dir.\n```\n要修改PPOCRLabel.py中 params 变量\n```python\n# \"text_detection_model_name\": \"PP-OCRv5_server_det\",\n\"text_detection_model_name\": \"PP-OCRv4_server_det\",\n# \"text_recognition_model_name\": \"PP-OCRv5_server_rec\",\n\"text_recognition_model_name\": \"PP-OCRv4_server_rec\",\n```",
        "state": "closed",
        "user": "haotianmingyue",
        "closed_by": "GreatV",
        "created_at": "2025-06-13T03:05:59+00:00",
        "updated_at": "2025-07-03T13:30:40+00:00",
        "closed_at": "2025-07-03T13:30:40+00:00",
        "comments_count": [
            "GreatV"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 171,
        "title": "环境问题",
        "body": "能不能出一个requirements.txt文件，说一下需要的包的版本，因为这个问题弄了两天了\n",
        "state": "closed",
        "user": "lltzahdz",
        "closed_by": "GreatV",
        "created_at": "2025-06-13T09:06:07+00:00",
        "updated_at": "2025-07-03T13:30:25+00:00",
        "closed_at": "2025-07-03T13:30:25+00:00",
        "comments_count": [
            "GreatV",
            "lltzahdz",
            "aymennasri",
            "lltzahdz",
            "GreatV",
            "Sam-gsj",
            "tsung-sc",
            "GreatV"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 174,
        "title": "RectBox re-regocnition crashes the app",
        "body": "Choosing re recognition of rectBox (`Ctrl+R`), crashes the app with this error: \n```\nTraceback (most recent call last):\n  File \"/home/aymen/Desktop/Python/dataset_annot/PPOCRLabel/PPOCRLabel.py\", line 3031, in singleRerecognition\n    storage = [(result[\"rec_text\"], result[\"rec_score\"])]\nKeyError: 'rec_text'\n\n\n--------------------------------------\nC++ Traceback (most recent call last):\n--------------------------------------\nNo stack trace in paddle, may be caused by external reasons.\n\n----------------------\nError Message Summary:\n----------------------\nFatalError: `Process abort signal` is detected by the operating system.\n  [TimeInfo: *** Aborted at 1749815728 (unix time) try \"date -d @1749815728\" if you are using GNU date ***]\n  [SignalInfo: *** SIGABRT (@0x3e800004dc3) received by PID 19907 (TID 0x7408db12f740) from PID 19907 ***]\n```",
        "state": "closed",
        "user": "aymennasri",
        "closed_by": "GreatV",
        "created_at": "2025-06-13T11:57:22+00:00",
        "updated_at": "2025-06-14T18:47:24+00:00",
        "closed_at": "2025-06-14T10:43:21+00:00",
        "comments_count": [
            "GreatV",
            "aymennasri",
            "GreatV",
            "Sam-gsj",
            "aymennasri",
            "Sam-gsj",
            "aymennasri",
            "Sam-gsj",
            "aymennasri"
        ],
        "labels": [
            "bug"
        ]
    },
    {
        "error": "'NoneType' object has no attribute 'repository'",
        "issue_number": 184
    },
    {
        "error": "'NoneType' object has no attribute 'repository'",
        "issue_number": 186
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 180,
        "title": "Auto-recognition on a single image crashes the app",
        "body": "Not always the case but this is what happens after doing auto-recognition on a single image directly after opening the application:\n```\n--------------------------------------\nC++ Traceback (most recent call last):\n--------------------------------------\n0   paddle::AnalysisPredictor::ZeroCopyRun(bool)\n1   paddle::framework::NaiveExecutor::RunInterpreterCore(std::vector<std::string, std::allocator<std::string > > const&, bool, bool)\n2   paddle::framework::InterpreterCore::Run(std::vector<std::string, std::allocator<std::string > > const&, bool, bool, bool, bool)\n3   paddle::framework::PirInterpreter::Run(std::vector<std::string, std::allocator<std::string > > const&, bool, bool, bool, bool)\n4   paddle::framework::PirInterpreter::TraceRunImpl()\n5   paddle::framework::PirInterpreter::TraceRunInstructionList(std::vector<std::unique_ptr<paddle::framework::InstructionBase, std::default_delete<paddle::framework::InstructionBase> >, std::allocator<std::unique_ptr<paddle::framework::InstructionBase, std::default_delete<paddle::framework::InstructionBase> > > > const&)\n6   paddle::framework::PirInterpreter::RunInstructionBase(paddle::framework::InstructionBase*)\n7   paddle::framework::PhiKernelInstruction::Run()\n8   phi::KernelImpl<void (*)(phi::CPUContext const&, phi::DenseTensor const&, phi::DenseTensor const&, std::vector<int, std::allocator<int> > const&, std::vector<int, std::allocator<int> > const&, std::string const&, std::vector<int, std::allocator<int> > const&, int, std::string const&, phi::DenseTensor*), &(void phi::ConvKernel<float, phi::CPUContext>(phi::CPUContext const&, phi::DenseTensor const&, phi::DenseTensor const&, std::vector<int, std::allocator<int> > const&, std::vector<int, std::allocator<int> > const&, std::string const&, std::vector<int, std::allocator<int> > const&, int, std::string const&, phi::DenseTensor*))>::Compute(phi::KernelContext*)\n9   void phi::ConvKernelImpl<float, phi::CPUContext>(phi::CPUContext const&, phi::DenseTensor const&, phi::DenseTensor const&, std::vector<int, std::allocator<int> > const&, std::vector<int, std::allocator<int> > const&, std::string const&, int, std::vector<int, std::allocator<int> > const&, std::string const&, phi::DenseTensor*)\n10  phi::funcs::Im2ColFunctor<(phi::funcs::ColFormat)0, phi::CPUContext, float>::operator()(phi::CPUContext const&, phi::DenseTensor const&, std::vector<int, std::allocator<int> > const&, std::vector<int, std::allocator<int> > const&, std::vector<int, std::allocator<int> > const&, phi::DenseTensor*, common::DataLayout)\n\n----------------------\nError Message Summary:\n----------------------\nFatalError: `Segmentation fault` is detected by the operating system.\n  [TimeInfo: *** Aborted at 1750078422 (unix time) try \"date -d @1750078422\" if you are using GNU date ***]\n  [SignalInfo: *** SIGSEGV (@0x73fe544db380) received by PID 18926 (TID 0x7405f05ff6c0) from PID 1414378368 ***]\n```",
        "state": "closed",
        "user": "aymennasri",
        "closed_by": "GreatV",
        "created_at": "2025-06-16T12:55:08+00:00",
        "updated_at": "2025-07-03T13:30:16+00:00",
        "closed_at": "2025-07-03T13:30:16+00:00",
        "comments_count": [
            "Sam-gsj"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 195,
        "title": "表格标注完成后，导出表格标注，显示导入成功，但gt.txt文件没有写入内容。",
        "body": "\n### 问题展示\n<img width=\"1228\" height=\"544\" alt=\"Image\" src=\"https://github.com/user-attachments/assets/1e98bd8d-a106-4ffb-aa34-47872ad576cd\" />\n\n<img width=\"1155\" height=\"833\" alt=\"Image\" src=\"https://github.com/user-attachments/assets/2175c2ba-4959-4d4b-b8b5-33f1853a68a1\" />\n\n### 其他说明\nlabel.txt中内容完整，difficute属性默认值为false，可以正常导出。但是点击导出表格标注后，gt.txt文件内容为空。",
        "state": "closed",
        "user": "LT-07",
        "closed_by": "LT-07",
        "created_at": "2025-07-06T09:53:47+00:00",
        "updated_at": "2025-07-06T10:13:08+00:00",
        "closed_at": "2025-07-06T10:13:08+00:00",
        "comments_count": [],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 187,
        "title": "ImportError: cannot import name 'PPStructureV3' from 'paddleocr'",
        "body": "ImportError: cannot import name 'PPStructureV3' from 'paddleocr'",
        "state": "closed",
        "user": "onenerve",
        "closed_by": "GreatV",
        "created_at": "2025-06-23T17:02:34+00:00",
        "updated_at": "2025-07-03T13:29:57+00:00",
        "closed_at": "2025-07-03T13:29:57+00:00",
        "comments_count": [
            "GreatV"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 188,
        "title": "linux  ubuntu20.04报错无法启动",
        "body": "查了好久的issue,issue里面的方法都用过了 还是报错，这该怎么搞?\n\n![Image](https://github.com/user-attachments/assets/a77c4676-5fb5-4b5a-bf32-2a81acfc1d5f)",
        "state": "open",
        "user": "tsung-sc",
        "closed_by": null,
        "created_at": "2025-06-25T03:23:44+00:00",
        "updated_at": "2025-07-08T07:00:53+00:00",
        "closed_at": null,
        "comments_count": [
            "tsung-sc",
            "GreatV",
            "tsung-sc",
            "tsung-sc",
            "GreatV",
            "tsung-sc",
            "tsung-sc",
            "tsung-sc",
            "GreatV",
            "GreatV",
            "13691126988",
            "robinchm",
            "LT-07",
            "robinchm",
            "LT-07"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 191,
        "title": "EXE程序运行报错FileNotFoundError: [Errno 2] No such file or directory: 'C:\\\\Users\\\\Administrator\\\\AppData\\\\Local\\\\Temp\\\\_MEI144802\\\\paddlex\\\\.version'",
        "body": "无论是自己打包的程序，还是官方给出打包好的程序，启动都会报这个错误。\n环境：win10",
        "state": "open",
        "user": "oneao",
        "closed_by": null,
        "created_at": "2025-07-02T08:30:37+00:00",
        "updated_at": "2025-07-02T14:19:19+00:00",
        "closed_at": null,
        "comments_count": [
            "oneao",
            "oneao",
            "GreatV"
        ],
        "labels": [
            "bug"
        ]
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 189,
        "title": "导出的识别结果图片不对，是个斜的，怎么回事啊?",
        "body": "我原始图片是正的 标记框里的图片也是正的 导出识别结果的训练图片是个斜东西 怎么回事啊？",
        "state": "closed",
        "user": "tsung-sc",
        "closed_by": "GreatV",
        "created_at": "2025-07-01T05:40:50+00:00",
        "updated_at": "2025-07-03T13:25:18+00:00",
        "closed_at": "2025-07-03T13:25:18+00:00",
        "comments_count": [
            "tsung-sc",
            "GreatV",
            "tsung-sc",
            "GreatV"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 194,
        "title": "请问.exe版本的可以支持导入自己的模型吗？Linux版本导入图片后自动标注就报错",
        "body": null,
        "state": "open",
        "user": "13691126988",
        "closed_by": null,
        "created_at": "2025-07-04T02:02:13+00:00",
        "updated_at": "2025-07-04T02:44:52+00:00",
        "closed_at": null,
        "comments_count": [
            "GreatV"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 196,
        "title": "segmentation fault",
        "body": "我已经能够正常打开软件，但是对某张图片点击自动识别后程序会自动退出，报错信息如下：\n\n```\nPython[76393:14237530] error messaging the mach port for IMKCFRunLoopWakeUpReliable\nzsh: segmentation fault  PPOCRLabel --lang ch\n```\n\n以下是我的运行信息：\n- python: 3.9.6\n- paddle: 3.1.0\n- PPOCRLabel: 3.1.2",
        "state": "closed",
        "user": "Voil3t",
        "closed_by": "Voil3t",
        "created_at": "2025-07-07T06:38:24+00:00",
        "updated_at": "2025-07-07T08:43:55+00:00",
        "closed_at": "2025-07-07T08:43:55+00:00",
        "comments_count": [
            "Voil3t",
            "GreatV",
            "Voil3t",
            "GreatV",
            "Voil3t",
            "GreatV",
            "Voil3t"
        ],
        "labels": []
    },
    {
        "repo": "PFCCLab/PPOCRLabel",
        "number": 197,
        "title": "能否进行layout、图片等信息的标注",
        "body": "我看代码中使用了PPStructureV3流水线，但是识别结果中却只有文本信息，而没有PPStructureV3流水线本身能够识别出的表格、layout、图片等信息的标注，我尝试将代码中PPStructureV3对象的use_chart_recognition、use_region_detection等参数从False改为True，但是识别结果仍然不变，请问这个功能现在支持吗？如果支持的话我需要如何才能得到这些内容的识别结果？如果不支持的话我能否通过其他Paddle官方的数据标注软件来实现这一需求？",
        "state": "open",
        "user": "Voil3t",
        "closed_by": null,
        "created_at": "2025-07-07T08:43:34+00:00",
        "updated_at": "2025-07-07T11:31:04+00:00",
        "closed_at": null,
        "comments_count": [
            "GreatV"
        ],
        "labels": []
    }
]